diff -urN kdenetwork-3.5.7.orig/filesharing/advanced/kcm_sambaconf/share.ui kdenetwork-3.5.7/filesharing/advanced/kcm_sambaconf/share.ui
--- kdenetwork-3.5.7.orig/filesharing/advanced/kcm_sambaconf/share.ui	2005-09-10 12:20:31 +0400
+++ kdenetwork-3.5.7/filesharing/advanced/kcm_sambaconf/share.ui	2007-07-03 16:40:23 +0400
@@ -2559,14 +2559,6 @@
                                         <string>Ms&amp;dfs root</string>
                                     </property>
                                 </widget>
-                                <widget class="QCheckBox" row="8" column="0" rowspan="1" colspan="2">
-                                    <property name="name">
-                                        <cstring>msdfsProxyChk</cstring>
-                                    </property>
-                                    <property name="text">
-                                        <string>Msdfs pro&amp;xy</string>
-                                    </property>
-                                </widget>
                                 <spacer row="9" column="1">
                                     <property name="name">
                                         <cstring>Spacer15</cstring>
@@ -2664,6 +2656,34 @@
                                         <cstring>dontDescendEdit</cstring>
                                     </property>
                                 </widget>
+                                <widget class="QLabel" row="8" column="0">
+                                    <property name="name">
+                                        <cstring>TextLabel6_2_3</cstring>
+                                    </property>
+                                    <property name="text">
+                                        <string>Msdfs pro&amp;xy</string>
+                                    </property>
+                                    <property name="buddy" stdset="0">
+                                        <cstring>msdfsProxyEdit</cstring>
+                                    </property>
+                                    <property name="toolTip" stdset="0">
+                                        <string>set if you want this share to be a msdfs proxy. See Samba docs for more information</string>
+                                    </property>
+                                    <property name="whatsThis" stdset="0">
+                                        <string>set if you want this share to be a msdfs proxy. See Samba docs for more information</string>
+                                    </property>
+                                </widget>
+                                <widget class="QLineEdit" row="8" column="1">
+                                    <property name="name">
+                                        <cstring>msdfsProxyEdit</cstring>
+                                    </property>
+                                    <property name="toolTip" stdset="0">
+                                        <string>set if you want this share to be a msdfs proxy. See Samba docs for more information</string>
+                                    </property>
+                                    <property name="whatsThis" stdset="0">
+                                        <string>set if you want this share to be a msdfs proxy. See Samba docs for more information</string>
+                                    </property>
+                                </widget>
                             </grid>
                         </widget>
                     </widget>
@@ -2953,7 +2973,7 @@
     <tabstop>setDirectoryChk</tabstop>
     <tabstop>fakeDirectoryCreateTimesChk</tabstop>
     <tabstop>msdfsRootChk</tabstop>
-    <tabstop>msdfsProxyChk</tabstop>
+    <tabstop>msdfsProxyEdit</tabstop>
     <tabstop>preexecEdit</tabstop>
     <tabstop>rootPreexecEdit</tabstop>
     <tabstop>postexecEdit</tabstop>
diff -urN kdenetwork-3.5.7.orig/filesharing/advanced/kcm_sambaconf/sharedlgimpl.cpp kdenetwork-3.5.7/filesharing/advanced/kcm_sambaconf/sharedlgimpl.cpp
--- kdenetwork-3.5.7.orig/filesharing/advanced/kcm_sambaconf/sharedlgimpl.cpp	2005-09-10 12:20:31 +0400
+++ kdenetwork-3.5.7/filesharing/advanced/kcm_sambaconf/sharedlgimpl.cpp	2007-07-03 16:46:04 +0400
@@ -249,7 +249,7 @@
   _dictMngr->add("fake directory create times",fakeDirectoryCreateTimesChk);
     
   _dictMngr->add("msdfs root",msdfsRootChk);
-  _dictMngr->add("msdfs proxy",msdfsProxyChk);
+  _dictMngr->add("msdfs proxy",msdfsProxyEdit);
   
   _dictMngr->load( _share );
   
