--- kdenetwork-040424/kopete/protocols/configure.in.in.orig	2004-03-13 18:47:15.000000000 +0100
+++ kdenetwork-040424/kopete/protocols/configure.in.in	2004-04-28 00:40:45.740315608 +0200
@@ -24,7 +24,7 @@
 ac_save_CXXFLAGS="$CXXFLAGS"
 LIBS="$LIBGG_LIBS -lgadu $LIBPTHREAD"
 CFLAGS="$CFLAGS $LIBGG_INCLUDES"
-AC_MSG_CHECKING([libgadu version 1.4rc2 to 1.4 with pthread support])
+AC_MSG_CHECKING([libgadu version 1.4rc2 to 1.5 with pthread support])
 	AC_TRY_RUN(
 	[
 	
@@ -44,7 +44,7 @@
 			return 1;
 		}
 
-		if ( min > 4 ){
+		if ( min > 5 ){
 			return 1;
 		}
 			     
