--- kdenetwork-3.0.3/libkdenetwork/kpgpui.h--	2002-08-28 17:35:12.000000000 +0200
+++ kdenetwork-3.0.3/libkdenetwork/kpgpui.h	2002-08-28 17:35:28.000000000 +0200
@@ -103,7 +103,7 @@ class KeySelectionDialog: public KDialog
                         const int acceptedKeys = 0,
                         QWidget *parent=0, const char *name=0,
                         bool modal=true );
-    virtual ~KeySelectionDialog() {};
+    virtual ~KeySelectionDialog();
 
     virtual QCString key(void) const {return mKeyID;};
     virtual bool rememberSelection() const { if (mRememberCB)
--- kdenetwork-3.0.3/libkdenetwork/kpgpui.cpp--	2002-08-28 17:34:06.000000000 +0200
+++ kdenetwork-3.0.3/libkdenetwork/kpgpui.cpp	2002-08-28 17:35:05.000000000 +0200
@@ -266,6 +266,12 @@ KeySelectionDialog::KeySelectionDialog( 
            this,      SLOT( slotSelectionChanged( QListViewItem * ) ) );
 }
 
+KeySelectionDialog::~KeySelectionDialog()
+{
+  delete keyGood;
+  delete keyBad;
+  delete keyUnknown;
+}
 
 void KeySelectionDialog::initKeylist( const KeyList& keyList,
                                       const QCString& keyId )
