--- kdenetwork-3.2.0/dcoprss/document.cpp~	2004-01-29 07:27:06.453723968 +0100
+++ kdenetwork-3.2.0/dcoprss/document.cpp	2004-01-29 07:27:37.742967280 +0100
@@ -76,7 +76,7 @@
 			connect(img, SIGNAL(gotPixmap(const QPixmap &)),
 				SLOT(pixmapLoaded(const QPixmap &)));
 			img->getPixmap();
-			pixmapUpdateing(DCOPRef(this));
+			pixmapUpdating(DCOPRef(this));
 		}
 		m_isLoading = false;
 		documentUpdated(DCOPRef(this));
@@ -246,7 +246,7 @@
 		Loader *loader = Loader::create(this,
 			SLOT(loadingComplete(Loader *, Document, Status)));
 		loader->loadFrom(m_Url,  new FileRetriever());
-		documentUpdateing(DCOPRef(this));
+		documentUpdating(DCOPRef(this));
 	}
 	else
 	{
@@ -263,7 +263,7 @@
 				connect(img, SIGNAL(gotPixmap(const QPixmap &)),
 					SLOT(pixmapLoaded(const QPixmap &)));
 				img->getPixmap();
-				pixmapUpdateing(DCOPRef(this));
+				pixmapUpdating(DCOPRef(this));
 			}
 		}
 		*/
--- kdenetwork-3.2.0/dcoprss/service.h~	2003-06-18 14:27:52.000000000 +0200
+++ kdenetwork-3.2.0/dcoprss/service.h	2004-01-29 07:30:22.186967992 +0100
@@ -132,7 +132,7 @@
 		/**
 		* The pixmap is currently loading
 		**/
-		void pixmapUpdateing(DCOPRef);
+		void pixmapUpdating(DCOPRef);
 		/**
 		*  The pixmap is ready for viewing
 		*  you can then use dcopref->call("pixmap()"); to return it.
@@ -140,9 +140,9 @@
 		**/
 		void pixmapUpdated(DCOPRef);
 		/**
-		* The document is currently updateing
+		* The document is currently updating
 		**/
-		void documentUpdateing(DCOPRef);
+		void documentUpdating(DCOPRef);
 		/**
 		* The document is ready for viewing
 		* you can then use dcopref->call() to access its data
