diff -urN kdenetwork-3.5.5.org/kopete/protocols/meanwhile/meanwhileprotocol.cpp kdenetwork-3.5.5/kopete/protocols/meanwhile/meanwhileprotocol.cpp
--- kdenetwork-3.5.5.org/kopete/protocols/meanwhile/meanwhileprotocol.cpp	2006-10-01 19:26:54.000000000 +0200
+++ kdenetwork-3.5.5/kopete/protocols/meanwhile/meanwhileprotocol.cpp	2007-01-06 00:00:01.057496500 +0100
@@ -118,9 +118,4 @@
     return statusAccountOffline;
 }
 
-const Kopete::OnlineStatus MeanwhileProtocol::lookupStatus(
-            enum Kopete::OnlineStatusManager::Categories cats)
-{
-    return Kopete::OnlineStatusManager::self()->onlineStatus(this, cats);
-}
 #include "meanwhileprotocol.moc"
diff -urN kdenetwork-3.5.5.org/kopete/protocols/meanwhile/meanwhileprotocol.h kdenetwork-3.5.5/kopete/protocols/meanwhile/meanwhileprotocol.h
--- kdenetwork-3.5.5.org/kopete/protocols/meanwhile/meanwhileprotocol.h	2006-10-01 19:26:54.000000000 +0200
+++ kdenetwork-3.5.5/kopete/protocols/meanwhile/meanwhileprotocol.h	2007-01-05 23:59:47.936676500 +0100
@@ -59,9 +59,6 @@
 
     const Kopete::OnlineStatus accountOfflineStatus();
 
-    const Kopete::OnlineStatus lookupStatus(
-            enum Kopete::OnlineStatusManager::Categories cats);
-
     const Kopete::OnlineStatus statusOffline;
     const Kopete::OnlineStatus statusOnline;
     const Kopete::OnlineStatus statusAway;
