--- kdenetwork-3.0.4/kdict/applet/Makefile.am~	Mon Nov 11 18:23:44 2002
+++ kdenetwork-3.0.4/kdict/applet/Makefile.am	Mon Nov 11 18:23:44 2002
@@ -12,7 +12,7 @@
 
 EXTRA_DIST = $(lnk_DATA)
 
-kdictapplet_la_LDFLAGS = $(all_libraries) -version-info 1:0:0 -module -no-undefined
+kdictapplet_la_LDFLAGS = $(all_libraries) -module -avoid-version -no-undefined
 kdictapplet_la_LIBADD = $(LIB_KSYCOCA) $(LIB_KDEUI)
 
 messages: rc.cpp
--- kdenetwork-3.0.4/kdict/applet/Makefile.in~	Mon Nov 11 18:24:57 2002
+++ kdenetwork-3.0.4/kdict/applet/Makefile.in	Mon Nov 11 18:24:57 2002
@@ -247,7 +247,7 @@
 
 EXTRA_DIST = $(lnk_DATA)
 
-kdictapplet_la_LDFLAGS = $(all_libraries) -version-info 1:0:0 -module -no-undefined
+kdictapplet_la_LDFLAGS = $(all_libraries) -module -avoid-version -no-undefined 
 kdictapplet_la_LIBADD = $(LIB_KSYCOCA) $(LIB_KDEUI)
 subdir = kdict/applet
 mkinstalldirs = $(SHELL) $(top_srcdir)/admin/mkinstalldirs
