--- kdenetwork-030618/dcoprss/service.h~	2003-06-14 11:48:45.000000000 +0200
+++ kdenetwork-030618/dcoprss/service.h	2003-06-19 02:16:25.000000000 +0200
@@ -27,11 +27,11 @@
 #include <qstring.h>
 #include <qdatetime.h>
 #include <qpixmap.h>
-#include <rss/global.h>
-#include <rss/loader.h>
-#include <rss/document.h>
-#include <rss/article.h>
-#include <rss/image.h>
+#include <librss/global.h>
+#include <librss/loader.h>
+#include <librss/document.h>
+#include <librss/article.h>
+#include <librss/image.h>
 /**
 * This is a DCOP Service do not include this header in anything
 *
