--- kdenetwork-2.2.2/ktalkd/ktalkdlg/Makefile.am~	Fri Dec  7 10:17:22 2001
+++ kdenetwork-2.2.2/ktalkd/ktalkdlg/Makefile.am	Fri Dec  7 10:17:22 2001
@@ -3,8 +3,6 @@
 
 EXTRA_DIST	 =	
 
-if KDE_INSTALLED
-
 bin_PROGRAMS	 =	ktalkdlg
 ktalkdlg_SOURCES =	ktalkdlg.cpp
 
@@ -12,8 +10,6 @@
 LDFLAGS		=	$(all_libraries) $(KDE_RPATH)
 LDADD		=	$(LIB_KDEUI)
 
-endif
-
 #for extra warnings during compilation :
 #AM_CXXFLAGS = -ansi -pedantic -D_POSIX_SOURCE -D_BSD_SOURCE
 
