diff -urN kdenetwork-3.0.1.orig/kmail/kmmainwin.rc kdenetwork-3.0.1/kmail/kmmainwin.rc
--- kdenetwork-3.0.1.orig/kmail/kmmainwin.cpp	Wed May  8 20:29:53 2002
+++ kdenetwork-3.0.1/kmail/kmmainwin.cpp	Fri Jun 14 18:37:08 2002
@@ -1954,7 +1954,7 @@
   (void) new KAction( i18n("&Find in Message..."), KStdAccel::shortcut(KStdAccel::Find), this,
 		      SLOT(slotFind()), actionCollection(), "find_in_messages" );
 
-  (void) new KAction( i18n("Select &All Messages"), Key_K, this,
+  (void) new KAction( i18n("Select &All Messages"), "ok", Key_K, this,
 		      SLOT(slotMarkAll()), actionCollection(), "mark_all_messages" );
 
   (void) new KAction( i18n("Select Message &Text"), KStdAccel::shortcut(KStdAccel::SelectAll), this,
@@ -1982,10 +1982,10 @@
   removeFolderAction = new KAction( i18n("&Remove"), 0, this,
 		      SLOT(slotRemoveFolder()), actionCollection(), "remove" );
 
-  preferHtmlAction = new KToggleAction( i18n("Pre&fer HTML to Plain Text"), 0, this,
+  preferHtmlAction = new KToggleAction( i18n("Pre&fer HTML to Plain Text"),"html", 0, this,
 		      SLOT(slotOverrideHtml()), actionCollection(), "prefer_html" );
 
-  threadMessagesAction = new KToggleAction( i18n("&Thread Messages"), 0, this,
+  threadMessagesAction = new KToggleAction( i18n("&Thread Messages"),"view_tree", 0, this,
 		      SLOT(slotOverrideThread()), actionCollection(), "thread_messages" );
 
   //----- Message Menu
@@ -2192,7 +2192,7 @@
   (void) new KAction( i18n("Appl&y Filters"), "filter", CTRL+Key_J, this,
 		      SLOT(slotApplyFilters()), actionCollection(), "apply_filters" );
 
-  viewSourceAction = new KAction( i18n("&View Source..."), Key_V, this,
+  viewSourceAction = new KAction( i18n("&View Source..."), "source", Key_V, this,
 		      SLOT(slotShowMsgSrc()), actionCollection(), "view_source" );
 
 
@@ -2219,7 +2219,7 @@
   mViewMenu->setItemChecked((int)mMsgView->attachmentStyle()+5, TRUE);
 
   mViewMenu->insertSeparator();
-  toggleFixFontAction = new KToggleAction( i18n("Fixed Font &Widths"),
+  toggleFixFontAction = new KToggleAction( i18n("Fixed Font &Widths"),"text",
 			0, this, SLOT(slotToggleFixedFont()),
 			actionCollection(), "toggle_fixedfont" );
   viewMenuAction->insert( toggleFixFontAction );
diff -urN kdenetwork-3.0.1.orig/kmail/kmcomposewin.cpp kdenetwork-3.0.1/kmail/kmcomposewin.cpp
--- kdenetwork-3.0.1.orig/kmail/kmcomposewin.cpp	Wed May  8 20:29:49 2002
+++ kdenetwork-3.0.1/kmail/kmcomposewin.cpp	Fri Jun 14 17:58:16 2002
@@ -790,7 +790,7 @@
   (void) new KAction (i18n("Cl&ean Spaces"), 0, this, SLOT(slotCleanSpace()),
                       actionCollection(), "clean_spaces");

-  fixedFontAction = new KToggleAction( i18n("Fixed Font &Widths"), 0, this,
+  fixedFontAction = new KToggleAction( i18n("Fixed Font &Widths"), "text", 0, this,
                       SLOT(slotUpdateFont()), actionCollection(), "toggle_fixedfont" );

   //these are checkable!!!
