--- kdenetwork/krfb/krfb/krfb.desktop~	2003-09-06 13:31:55.000000000 +0200
+++ kdenetwork/krfb/krfb/krfb.desktop	2003-09-06 13:32:42.000000000 +0200
@@ -51,4 +51,4 @@
 Name[zh_CN]=桌面共享
 Name[zh_TW]=桌面分享
 Name[zu]=Ukwahlulelana kwe-Desktop
-Categories=Qt;KDE;System
+Categories=Qt;KDE;Network;
--- kdenetwork/kopete/kopete/kopete.desktop.orig	2004-03-13 18:46:53.000000000 +0100
+++ kdenetwork/kopete/kopete/kopete.desktop	2004-03-13 19:07:41.923725848 +0100
@@ -54,5 +54,5 @@
 Type=Application
 Terminal=0
 Icon=kopete
-Categories=Qt;KDE;Network
+Categories=Qt;KDE;Network;InstantMessaging;
 X-DCOP-ServiceType=Unique
