diff -urN -x CVS kdenetwork.orig/filesharing/advanced/kcm_sambaconf/kcmsambaconf.desktop kdenetwork/filesharing/advanced/kcm_sambaconf/kcmsambaconf.desktop
--- kdenetwork.orig/filesharing/advanced/kcm_sambaconf/kcmsambaconf.desktop	Fri Oct  1 15:54:28 2004
+++ kdenetwork/filesharing/advanced/kcm_sambaconf/kcmsambaconf.desktop	Mon Oct  4 15:25:42 2004
@@ -37,7 +37,7 @@
 Comment[sk]=KcmSambaConf - ovládací modul pre nastavenie serveru samba
 Comment[sl]=KcmSambaConf - Modul za KControl za nastavljanje strežnika Samba
 Comment[sr]=KcmSambaConf — модула KControl-а за подешавање Samba сервера
-Comment[sr@Latn]=KcmSambaConf — модула KControl-а за подешавање Samba сервера
+Comment[sr@Latn]=KcmSambaConf — modula KControl-a za podešavanje Samba servera
 Comment[sv]=IM SAMBA-inställning: En modul i inställningscentralen för att anpassa en SAMBA-server
 Comment[ta]=KcmSambaConf - ஒரு KControl கூறு சம்பா சேவையகத்தை உள்ளமைக்க உதவும்
 Comment[tg]=KcmSambaConf - воҳиди KИдора барои батанзимдарории хидматрасони samba
@@ -75,7 +75,7 @@
 Name[sk]=Nastavenie Samba
 Name[sl]=Nastavitve Sambe
 Name[sr]=Подешавање Samba-е
-Name[sr@Latn]=Подешавање Samba-е
+Name[sr@Latn]=Podešavanje Samba-e
 Name[sv]=Anpassa SAMBA
 Name[tg]=Батанзимдарории Samba
 Name[uk]=Налаштування Samba
diff -urN -x CVS kdenetwork.orig/filesharing/advanced/propsdlgplugin/fileshare_propsdlgplugin.desktop kdenetwork/filesharing/advanced/propsdlgplugin/fileshare_propsdlgplugin.desktop
--- kdenetwork.orig/filesharing/advanced/propsdlgplugin/fileshare_propsdlgplugin.desktop	Thu Sep  9 15:55:27 2004
+++ kdenetwork/filesharing/advanced/propsdlgplugin/fileshare_propsdlgplugin.desktop	Mon Oct  4 15:25:42 2004
@@ -23,7 +23,7 @@
 Name[sk]=Nastavenie zdieľania priečinkov pre Konqueror
 Name[sl]=Stran z lastnostmi za souporabo mape
 Name[sr]=Страна Konqueror-а са својствима директоријума за дељење фајлова
-Name[sr@Latn]=Страна Konqueror-а са својствима директоријума за дељење фајлова
+Name[sr@Latn]=Strana Konqueror-a sa svojstvima direktorijuma za deljenje fajlova
 Name[sv]=Konqueror fildelningssida med katalogegenskaper
 Name[ta]=Fileshare Konqueror அடைவு பண்புகளின் பக்கம்
 Name[tg]=Саҳифаи Феҳристи Хусусиятҳо оиди Konqueror Истифодабарии Муштараки Файлҳо
@@ -49,7 +49,7 @@
 Comment[sk]=Modul Konquerora pre zdieľanie priečinku v lokálnej sieti
 Comment[sl]=Vstavek za Konqueror s pogovornim oknom za lastnosti za souporabo mape v krajevnem omrežju
 Comment[sr]=Прикључак Konqueror-а за подешавање својстава дељења директоријума у локалној мрежи
-Comment[sr@Latn]=Прикључак Konqueror-а за подешавање својстава дељења директоријума у локалној мрежи
+Comment[sr@Latn]=Priključak Konqueror-a za podešavanje svojstava deljenja direktorijuma u lokalnoj mreži
 Comment[sv]=Konqueror-insticksprogram med egenskapsdialogruta för att dela en katalog i det lokala nätverket
 Comment[ta]=Konqueror பண்புகளின் உரையாடல் செருகுகள் அடைவை சம்பா சேவையகத்துடன் பகிரவேண்டிய செருகுகள்
 Comment[tg]=Модули муколамаи хусусиятҳои Konqueror барои истифодабарии муштараки феҳрист бо шабакаи маҳалллӣ
diff -urN -x CVS kdenetwork.orig/filesharing/simple/fileshare.desktop kdenetwork/filesharing/simple/fileshare.desktop
--- kdenetwork.orig/filesharing/simple/fileshare.desktop	Thu Sep  9 15:55:28 2004
+++ kdenetwork/filesharing/simple/fileshare.desktop	Mon Oct  4 15:25:42 2004
@@ -32,7 +32,7 @@
 Name[sk]=Zdieľanie súborov
 Name[sl]=Souporaba datotek
 Name[sr]=Дељење фајлова
-Name[sr@Latn]=Дељење фајлова
+Name[sr@Latn]=Deljenje fajlova
 Name[sv]=Fildelning
 Name[ta]=கோப்பு பகிர்வு
 Name[tg]=Истифодабарии муштараки файлҳо
@@ -61,7 +61,7 @@
 Comment[sk]=Povolenie alebo zakázanie zdieľania súborov
 Comment[sl]=Omogoči ali onemogoči souporabo datotek
 Comment[sr]=Укључи или искључи дељење фајлова
-Comment[sr@Latn]=Укључи или искључи дељење фајлова
+Comment[sr@Latn]=Uključi ili isključi deljenje fajlova
 Comment[sv]=Aktivera eller inaktivera fildelning
 Comment[ta]=கோப்பு பகிர்வை செயல்படுத்து அல்லது செயல்நீக்கு
 Comment[tg]=Фаъол сохтан ё хомӯш кардани истифобадарии муштараки файлҳо
@@ -90,7 +90,7 @@
 Keywords[sk]=zdieľaný disk,share
 Keywords[sl]=Souporaba
 Keywords[sr]=Share,дељење
-Keywords[sr@Latn]=Share,дељење
+Keywords[sr@Latn]=Share,deljenje
 Keywords[sv]=Dela
 Keywords[ta]=பகிர்வு
 Keywords[tg]=Муштаракан доштан
diff -urN -x CVS kdenetwork.orig/knewsticker/eventsrc kdenetwork/knewsticker/eventsrc
--- kdenetwork.orig/knewsticker/eventsrc	Tue Aug 24 08:30:59 2004
+++ kdenetwork/knewsticker/eventsrc	Tue Oct  5 15:20:35 2004
@@ -62,6 +62,7 @@
 Name[ar]=توجد أخبار جديدة
 Name[az]=Mövcud Yeni Xəbərlər
 Name[bg]=Пристигнаха нови новини
+Name[br]=Keleier nevez da gaout
 Name[bs]=Nove vijesti su dostupne
 Name[ca]=Noves notícies disponibles
 Name[cs]=Jsou dostupné nové zprávy
diff -urN -x CVS kdenetwork.orig/knewsticker/knewsticker.cpp kdenetwork/knewsticker/knewsticker.cpp
--- kdenetwork.orig/knewsticker/knewsticker.cpp	Thu Apr 29 13:06:18 2004
+++ kdenetwork/knewsticker/knewsticker.cpp	Sun Oct  3 22:44:36 2004
@@ -461,7 +461,7 @@
 		insertSeparator();
 
 	insertItem(lookIcon, i18n("Check News"), m_parent, SLOT(slotUpdateNews()));
-	int i = insertItem(i18n("Offline Mode"), this, SLOT(slotToggleOfflineMode()), 4711 );
+	int i = insertItem(i18n("Offline Mode"), this, SLOT(slotToggleOfflineMode()), 0, 4711 );
 	setItemChecked(i, m_parent->m_cfg->offlineMode());
 
 	if (m_fullMenu) {
diff -urN -x CVS kdenetwork.orig/knewsticker/knewstickerstub/knewstickerstub.desktop kdenetwork/knewsticker/knewstickerstub/knewstickerstub.desktop
--- kdenetwork.orig/knewsticker/knewstickerstub/knewstickerstub.desktop	Tue Aug 31 09:22:22 2004
+++ kdenetwork/knewsticker/knewstickerstub/knewstickerstub.desktop	Mon Oct  4 15:25:44 2004
@@ -22,7 +22,7 @@
 Name[sk]=Nastavenie KNewsTicker
 Name[sl]=Nastavitveni vmesnik programa za prikazovanje novic
 Name[sr]=Интерфејс за подешавање KNewsTicker-а
-Name[sr@Latn]=Интерфејс за подешавање KNewsTicker-а
+Name[sr@Latn]=Interfejs za podešavanje KNewsTicker-a
 Name[sv]=Gränssnitt för anpassning av nyhetsövervakaren
 Name[ta]=கேசெய்தி அறிவிப்பான் கட்டமை முன்னிலை
 Name[tg]=KПешохири Танзими Дидабони Ахборот
diff -urN -x CVS kdenetwork.orig/kopete/kopete/config/accounts/kopete_accountconfig.desktop kdenetwork/kopete/kopete/config/accounts/kopete_accountconfig.desktop
--- kdenetwork.orig/kopete/kopete/config/accounts/kopete_accountconfig.desktop	Tue Aug 24 08:31:06 2004
+++ kdenetwork/kopete/kopete/config/accounts/kopete_accountconfig.desktop	Sun Oct  3 15:37:41 2004
@@ -13,6 +13,7 @@
 Name=Accounts
 Name[be]=Рахункі
 Name[bg]=Сметки
+Name[br]=Kontoù
 Name[bs]=Računi
 Name[ca]=Comptes
 Name[cs]=Účty
diff -urN -x CVS kdenetwork.orig/kopete/kopete/config/plugins/kopetepluginconfig.cpp kdenetwork/kopete/kopete/config/plugins/kopetepluginconfig.cpp
--- kdenetwork.orig/kopete/kopete/config/plugins/kopetepluginconfig.cpp	Thu Jul 22 00:04:08 2004
+++ kdenetwork/kopete/kopete/config/plugins/kopetepluginconfig.cpp	Mon Oct 18 13:24:15 2004
@@ -34,6 +34,11 @@
 	bool isChanged;
 };
 
+KopetePluginConfig::~KopetePluginConfig()
+{
+	delete d;
+}
+
 KopetePluginConfig::KopetePluginConfig( QWidget *parent, const char *name )
 : KDialogBase( Plain, i18n( "Configure" ), Help | Default | Cancel | Apply | Ok | User1,
 	Ok, parent, name, false, true, KGuiItem( i18n( "&Reset" ), "undo" ) )
diff -urN -x CVS kdenetwork.orig/kopete/kopete/config/plugins/kopetepluginconfig.h kdenetwork/kopete/kopete/config/plugins/kopetepluginconfig.h
--- kdenetwork.orig/kopete/kopete/config/plugins/kopetepluginconfig.h	Sat Oct 18 00:01:42 2003
+++ kdenetwork/kopete/kopete/config/plugins/kopetepluginconfig.h	Mon Oct 18 13:24:15 2004
@@ -33,7 +33,7 @@
 
 public:
 	KopetePluginConfig( QWidget *parent, const char *name = 0L );
-
+	~KopetePluginConfig();
 	void apply();
 
 public slots:
diff -urN -x CVS kdenetwork.orig/kopete/kopete/eventsrc kdenetwork/kopete/kopete/eventsrc
--- kdenetwork.orig/kopete/kopete/eventsrc	Tue Aug 31 09:22:28 2004
+++ kdenetwork/kopete/kopete/eventsrc	Mon Oct  4 15:25:49 2004
@@ -22,7 +22,7 @@
 Comment[pt_BR]=Mensageiro Kopete
 Comment[sl]=Sporočilnik Kopete
 Comment[sr]=Гласник Kopete
-Comment[sr@Latn]=Kopete glasnik
+Comment[sr@Latn]=Glasnik Kopete
 Comment[sv]=Kopete meddelandeklient
 Comment[ta]=உடனடி தூதர்
 Comment[tg]=Пайёмбари Kopete
@@ -95,7 +95,7 @@
 Comment[sk]=Zvukové upozornenia pre udalosti Kopete
 Comment[sl]=Zvočna obvestila o dogodkih v Kopete
 Comment[sr]=Звучно обавештење о Kopete-овим догађајима
-Comment[sr@Latn]=Звучно обавештење о Kopete-овим догађајима
+Comment[sr@Latn]=Zvučno obaveštenje o Kopete-ovim događajima
 Comment[sv]=Ljudunderrättelse för händelser i Kopete
 Comment[ta]= Kopete ல் ஒலி அறிவிப்பதற்கான நிகழ்வுகள்
 Comment[tg]=Огоҳонии овозӣ оиди ҳодисаҳои Kopete
@@ -170,7 +170,7 @@
 Comment[sk]=Príchod správy
 Comment[sl]=Prejeto je bilo prihajajoče sporočilo
 Comment[sr]=Примљена је долазећа порука
-Comment[sr@Latn]=Примљена је долазећа порука
+Comment[sr@Latn]=Primljena je dolazeća poruka
 Comment[sv]=Ett inkommande meddelande har anlänt
 Comment[ta]=உள்வரும் தகவல் பெறப்பட்டது
 Comment[tg]=Пайёми воридшаванда қабул гардид
@@ -245,7 +245,7 @@
 Comment[sk]=Odoslanie správy
 Comment[sl]=Poslano je bilo odhajajoče poročilo
 Comment[sr]=Одлазећа порука је послата
-Comment[sr@Latn]=Одлазећа порука је послата
+Comment[sr@Latn]=Odlazeća poruka je poslata
 Comment[sv]=Ett utgående meddelande har skickats
 Comment[ta]=வெளிச்செல்ல வேண்டிய தகவல் அனுப்பட்டது
 Comment[tg]=Пайёми хориҷшаванда фиристода шуд
@@ -313,7 +313,7 @@
 Comment[sk]=Používateľ je on-line
 Comment[sl]=Uporabnik je vzpostavil povezavo
 Comment[sr]=Корисник се повезао
-Comment[sr@Latn]=Корисник се повезао
+Comment[sr@Latn]=Korisnik se povezao
 Comment[sv]=En användare har kopplat upp
 Comment[ta]=பயனர் உரையாட வந்துள்ளார்
 Comment[tg]=Корванде ба шабака ворид шуд
@@ -380,7 +380,7 @@
 Comment[sk]=Používateľ je off-line
 Comment[sl]=Uporabnik je prekinil povezavo
 Comment[sr]=Корисник је отишао са везе
-Comment[sr@Latn]=Корисник је отишао са везе
+Comment[sr@Latn]=Korisnik je otišao sa veze
 Comment[sv]=En användare har kopplat ner
 Comment[ta]=பயனர் வெளியேறிவிட்டார்
 Comment[tg]=Корванде аз шабака хориҷ шуд
@@ -454,7 +454,7 @@
 Comment[sk]=Zmena stavu pripojenia používateľa
 Comment[sl]=Uporabnik je spremenil svoje stanje
 Comment[sr]=Корисник је променио свој статус на вези
-Comment[sr@Latn]=Корисник је променио свој статус на вези
+Comment[sr@Latn]=Korisnik je promenio svoj status na vezi
 Comment[sv]=Uppkopplingsstatus för en användare har ändrats
 Comment[ta]= இணைய உரையாடலில் பயனரின் நிலை மாற்றப்பட்டது
 Comment[tg]=Ҳолати алоқаҳои пайваста тағир дода шуданд
@@ -525,7 +525,7 @@
 Comment[sk]=Prijatá zvýraznená správa
 Comment[sl]=Prejeto je bilo osvetljeno sporočilo
 Comment[sr]=Примљена је истакнута порука
-Comment[sr@Latn]=Примљена је истакнута порука
+Comment[sr@Latn]=Primljena je istaknuta poruka
 Comment[sv]=Ett markerat meddelande har tagits emot
 Comment[ta]=தனிப்படுத்தப்பட்ட செய்தி பெறப்பட்டது
 Comment[tg]=Пайёмҳои равшаншаванда қабул гардиданд
@@ -558,7 +558,7 @@
 Name[sk]=Správy s nízkou prioritou
 Name[sl]=Manj pomembna sporočila
 Name[sr]=Поруке ниског приоритета
-Name[sr@Latn]=Поруке ниског приоритета
+Name[sr@Latn]=Poruke niskog prioriteta
 Name[sv]=Lågprioriterat meddelande
 Name[ta]=குறைந்த முன்னுரிமை உள்ள செய்திகள்
 Name[tg]=Пайёмҳои Имтиёзашон Кам
@@ -586,7 +586,7 @@
 Comment[sk]=Prijatá správa s nízkou prioritou
 Comment[sl]=Prejeto je bilo manj pomembno sporočilo
 Comment[sr]=Примљена је порука означена ниским приоритетом
-Comment[sr@Latn]=Примљена је порука означена ниским приоритетом
+Comment[sr@Latn]=Primljena je poruka označena niskim prioritetom
 Comment[sv]=Ett inkommande meddelande har anlänt
 Comment[ta]=எளிய முன்னுரிமை ஏற்கப்பட்ட செய்தி சுழற்சி
 Comment[tg]=Пайёми бо имтиёзи паст нишона карда шуда қабул гардид
@@ -611,7 +611,7 @@
 Name[ru]=Почта Yahoo
 Name[sk]=Pošta Yahoo
 Name[sr]=Yahoo пошта
-Name[sr@Latn]=Yahoo пошта
+Name[sr@Latn]=Yahoo pošta
 Name[sv]=Yahoo e-post
 Name[ta]=யாஹூ அஞ்சல்
 Name[tg]=Пости Yahoo
@@ -639,7 +639,7 @@
 Comment[sk]=Do vašej schránky Yahoo prišla nová správa
 Comment[sl]=Nova pošta je prispela v vaš nabiralnik Yahoo
 Comment[sr]=Нова порука је стигла у ваше Yahoo сандуче
-Comment[sr@Latn]=Нова порука је стигла у ваше Yahoo сандуче
+Comment[sr@Latn]=Nova poruka je stigla u vaše Yahoo sanduče
 Comment[sv]=Ett nytt brev har anlänt i din Yahoo-inkorg
 Comment[ta]=உங்கள் மின் அஞ்சல் பெட்டிக்குள் புதிய மின்னஞ்சல் ஒன்று வந்துள்ளது
 Comment[tg]=Ба қутии пости Yahoo-и шумо пайёми электронии нав омад
@@ -704,7 +704,7 @@
 Comment[sk]=Do vašej schránky MSN prišla nová správa
 Comment[sl]=Nova pošta je prispela v vaš nabiralnik MSN
 Comment[sr]=Нова порука је стигла у ваше MSN сандуче
-Comment[sr@Latn]=Нова порука је стигла у ваше MSN сандуче
+Comment[sr@Latn]=Nova poruka je stigla u vaše MSN sanduče
 Comment[sv]=Ett nytt brev har anlänt i din MSN-inkorg
 Comment[ta]=உங்கள் எம்எஸ்என் அஞ்சல் பெட்டிக்கு புதிய மின்னஞ்சல் வந்துள்ளது
 Comment[tg]=Ба қутии пости MSN-и шумо пайёми электронии нав омад
diff -urN -x CVS kdenetwork.orig/kopete/kopete/kopetewindow.cpp kdenetwork/kopete/kopete/kopetewindow.cpp
--- kdenetwork.orig/kopete/kopete/kopetewindow.cpp	Mon Aug 23 05:29:32 2004
+++ kdenetwork/kopete/kopete/kopetewindow.cpp	Tue Oct 19 02:33:48 2004
@@ -196,7 +196,7 @@
 
 	globalAccel = new KGlobalAccel( this );
 	globalAccel->insert( QString::fromLatin1("Read Message"), i18n("Read Message"), i18n("Read the next pending message"),
-		CTRL+SHIFT+Key_I, KKey::QtWIN+CTRL+Key_I, KopeteMessageManagerFactory::factory() , SIGNAL(readMessage()) );
+		CTRL+SHIFT+Key_I, KKey::QtWIN+CTRL+Key_I, KopeteMessageManagerFactory::factory(), SLOT(slotReadMessage()) );
 
 	globalAccel->insert( QString::fromLatin1("Show/Hide Contact List"), i18n("Show/Hide Contact List"), i18n("Show or hide the contact list"),
 		CTRL+SHIFT+Key_C, KKey::QtWIN+CTRL+Key_C, this, SLOT(slotShowHide()) );
@@ -400,7 +400,7 @@
 void KopeteWindow::slotConfToolbar()
 {
 	saveMainWindowSettings(KGlobal::config(), "General Options");
-	KEditToolbar *dlg = new KEditToolbar(actionCollection(), "kopeteui.rc");
+	KEditToolbar *dlg = new KEditToolbar(factory());
 	connect( dlg, SIGNAL(newToolbarConfig()), this, SLOT(slotUpdateToolbar()) );
 	connect( dlg, SIGNAL(finished()) , dlg, SLOT(deleteLater()));
 	dlg->show();
@@ -408,7 +408,6 @@
 
 void KopeteWindow::slotUpdateToolbar()
 {
-	createGUI("kopeteui.rc", false);
 	applyMainWindowSettings(KGlobal::config(), "General Options");
 }
 
diff -urN -x CVS kdenetwork.orig/kopete/kopete/x-kopete-emoticons.desktop kdenetwork/kopete/kopete/x-kopete-emoticons.desktop
--- kdenetwork.orig/kopete/kopete/x-kopete-emoticons.desktop	Tue Aug 31 09:22:28 2004
+++ kdenetwork/kopete/kopete/x-kopete-emoticons.desktop	Mon Oct  4 15:25:49 2004
@@ -30,7 +30,7 @@
 Comment[sk]=Archív smajlíkov Kopete
 Comment[sl]=Arhiv z ikonami čustev za kopete
 Comment[sr]=Kopete-ова архива емотикона
-Comment[sr@Latn]=Kopete-ова архива емотикона
+Comment[sr@Latn]=Kopete-ova arhiva emotikona
 Comment[sv]=Kopetes smilisarkiv
 Comment[ta]=Kopete எமோடிக்கான் வளைவு
 Comment[tg]=Бойгонии Kopete Emoticon 
diff -urN -x CVS kdenetwork.orig/kopete/libkopete/compat/kresolver.cpp kdenetwork/kopete/libkopete/compat/kresolver.cpp
--- kdenetwork.orig/kopete/libkopete/compat/kresolver.cpp	Sat Oct  2 06:36:30 2004
+++ kdenetwork/kopete/libkopete/compat/kresolver.cpp	Mon Oct 11 20:18:34 2004
@@ -567,8 +567,7 @@
     I18N_NOOP("requested service not supported for this socket type"), // UnsupportedService
     I18N_NOOP("requested socket type not supported"),	// UnsupportedSocketType
     I18N_NOOP("unknown error"),			// UnknownError
-    I18N_NOOP2("1: the i18n'ed system error code, from errno",
-	      "system error: %1")		// SystemError
+    I18N_NOOP("system error: %1")		// SystemError
   };
 
   // handle the special value
diff -urN -x CVS kdenetwork.orig/kopete/libkopete/compat/kresolvermanager.cpp kdenetwork/kopete/libkopete/compat/kresolvermanager.cpp
--- kdenetwork.orig/kopete/libkopete/compat/kresolvermanager.cpp	Mon Aug 23 05:46:46 2004
+++ kdenetwork/kopete/libkopete/compat/kresolvermanager.cpp	Mon Oct 11 20:18:34 2004
@@ -203,7 +203,7 @@
 // a thread will try maxThreadRetries to get data, waiting at most
 // maxThreadWaitTime milliseconds between each attempt. After that, it'll
 // exit
-static const int maxThreadWaitTime = 20000; // 20 seconds
+static const int maxThreadWaitTime = ULONG_MAX; // wait forever
 static const int maxThreads = 5;
 
 static pid_t pid;		// FIXME -- disable when everything is ok
diff -urN -x CVS kdenetwork.orig/kopete/libkopete/compat/ksockssocketdevice.cpp kdenetwork/kopete/libkopete/compat/ksockssocketdevice.cpp
--- kdenetwork.orig/kopete/libkopete/compat/ksockssocketdevice.cpp	Sat Oct  2 06:36:30 2004
+++ kdenetwork/kopete/libkopete/compat/ksockssocketdevice.cpp	Mon Oct 11 20:18:34 2004
@@ -169,7 +169,7 @@
   return new KSocksSocketDevice(newfd);
 }
 
-static int socks_read_common(int sockfd, char *data, Q_ULONG maxlen, KSocketAddress* from, ssize_t &retval, bool peek = false)
+static int socks_read_common(int sockfd, char *data, Q_ULONG maxlen, KNetwork::KSocketAddress* from, ssize_t &retval, bool peek = false)
 {
   kde_socklen_t len;
   if (from)
@@ -304,7 +304,7 @@
   return retval;
 }
 
-KSocketAddress KSocksSocketDevice::localAddress() const
+KNetwork::KSocketAddress KSocksSocketDevice::localAddress() const
 {
   if (m_sockfd == -1)
     return KSocketAddress();	// not open, empty value
@@ -333,7 +333,7 @@
   return localAddress;
 }
 
-KSocketAddress KSocksSocketDevice::peerAddress() const
+KNetwork::KSocketAddress KSocksSocketDevice::peerAddress() const
 {
   if (m_sockfd == -1)
     return KSocketAddress();	// not open, empty value
@@ -362,7 +362,7 @@
   return peerAddress;
 }
 
-KSocketAddress KSocksSocketDevice::externalAddress() const
+KNetwork::KSocketAddress KSocksSocketDevice::externalAddress() const
 {
   // return empty, indicating unknown external address
   return KSocketAddress();
diff -urN -x CVS kdenetwork.orig/kopete/libkopete/kopetecontactlist.cpp kdenetwork/kopete/libkopete/kopetecontactlist.cpp
--- kdenetwork.orig/kopete/libkopete/kopetecontactlist.cpp	Thu Jul 29 08:48:16 2004
+++ kdenetwork/kopete/libkopete/kopetecontactlist.cpp	Mon Oct 18 14:34:17 2004
@@ -912,9 +912,12 @@
 		setSelectedItems( d->selectedMetaContacts, d->selectedGroups );
 	}
 
-	d->groups.remove( g );
-	emit groupRemoved( g );
-	delete g;
+	if ( g->type() == KopeteGroup::Normal )
+	{
+		d->groups.remove( g );
+		emit groupRemoved( g );
+		delete g;
+	}
 }
 
 KopeteGroup * KopeteContactList::getGroup(const QString& displayName, int type)
diff -urN -x CVS kdenetwork.orig/kopete/libkopete/kopetegroup.cpp kdenetwork/kopete/libkopete/kopetegroup.cpp
--- kdenetwork.orig/kopete/libkopete/kopetegroup.cpp	Tue Jul 27 09:10:45 2004
+++ kdenetwork/kopete/libkopete/kopetegroup.cpp	Sun Oct 17 16:14:05 2004
@@ -96,6 +96,11 @@
 
 KopeteGroup::~KopeteGroup()
 {
+	if ( d->type == TopLevel )
+		s_topLevel = 0L;
+	if ( d->type == Temporary )
+		s_temporary = 0L;
+
 	delete d;
 }
 
diff -urN -x CVS kdenetwork.orig/kopete/libkopete/kopetemessagemanagerfactory.cpp kdenetwork/kopete/libkopete/kopetemessagemanagerfactory.cpp
--- kdenetwork.orig/kopete/libkopete/kopetemessagemanagerfactory.cpp	Sat Apr  3 17:06:54 2004
+++ kdenetwork/kopete/libkopete/kopetemessagemanagerfactory.cpp	Tue Oct 19 02:33:48 2004
@@ -105,6 +105,11 @@
 	return (result);
 }
 
+void KopeteMessageManagerFactory::slotReadMessage()
+{
+	emit readMessage();
+}
+
 void KopeteMessageManagerFactory::addKopeteMessageManager(KopeteMessageManager * result)
 {
 	if(result->mmId() == 0)
diff -urN -x CVS kdenetwork.orig/kopete/libkopete/kopetemessagemanagerfactory.h kdenetwork/kopete/libkopete/kopetemessagemanagerfactory.h
--- kdenetwork.orig/kopete/libkopete/kopetemessagemanagerfactory.h	Sat Jan 24 22:23:04 2004
+++ kdenetwork/kopete/libkopete/kopetemessagemanagerfactory.h	Tue Oct 19 02:33:48 2004
@@ -188,6 +188,9 @@
 	 */
 	 void getActiveView( KopeteView *& );
 
+public slots:
+	void slotReadMessage();
+
 private:
 	KopeteMessageManagerFactory( QObject* parent = 0, const char* name = 0 );
 
diff -urN -x CVS kdenetwork.orig/kopete/libkopete/kopeteplugin.desktop kdenetwork/kopete/libkopete/kopeteplugin.desktop
--- kdenetwork.orig/kopete/libkopete/kopeteplugin.desktop	Tue Aug 24 08:31:07 2004
+++ kdenetwork/kopete/libkopete/kopeteplugin.desktop	Mon Oct  4 15:25:49 2004
@@ -36,7 +36,7 @@
 Comment[sk]=Modul Kopete
 Comment[sl]=Dodatek za Kopete
 Comment[sr]=Прикључак за Kopete
-Comment[sr@Latn]=Dodatak za Kopete
+Comment[sr@Latn]=Priključak za Kopete
 Comment[sv]=Insticksprogram för Kopete
 Comment[ta]=Kopete செருகல்
 Comment[tg]=Модули Kopete 
diff -urN -x CVS kdenetwork.orig/kopete/libkopete/kopeteprotocol.desktop kdenetwork/kopete/libkopete/kopeteprotocol.desktop
--- kdenetwork.orig/kopete/libkopete/kopeteprotocol.desktop	Tue Aug 24 08:31:07 2004
+++ kdenetwork/kopete/libkopete/kopeteprotocol.desktop	Mon Oct  4 15:25:49 2004
@@ -35,7 +35,7 @@
 Comment[sk]=Modul protokolu Kopete
 Comment[sl]=Dodatek za protokol Kopete
 Comment[sr]=Kopete-ов прикључак за протокол
-Comment[sr@Latn]=Kopete-ov dodatak za protokol
+Comment[sr@Latn]=Kopete-ov priključak za protokol
 Comment[sv]=Protokollinsticksprogram för Kopete
 Comment[ta]=Kopete விதிமுறை செருகல்
 Comment[tg]=Модули Қарордоди Kopete 
diff -urN -x CVS kdenetwork.orig/kopete/libkopete/kopeteui.desktop kdenetwork/kopete/libkopete/kopeteui.desktop
--- kdenetwork.orig/kopete/libkopete/kopeteui.desktop	Tue Aug 24 08:31:07 2004
+++ kdenetwork/kopete/libkopete/kopeteui.desktop	Mon Oct  4 15:25:49 2004
@@ -34,7 +34,7 @@
 Comment[sk]=Modul rozhrania Kopete
 Comment[sl]=Dodatek za uporabniški vmesnik v Kopete
 Comment[sr]=Прикључак за Kopete-ов кориснички интерфејс
-Comment[sr@Latn]=Dodatak za Kopete-ov korisnički interfejs
+Comment[sr@Latn]=Priključak za Kopete-ov korisnički interfejs
 Comment[sv]=Gränssnittsinsticksprogram för Kopete
 Comment[ta]=ஒரு Kopete UI செருகல்
 Comment[tg]=Модули Интерфейси Корвандии Kopete
diff -urN -x CVS kdenetwork.orig/kopete/libkopete/ui/accountselector.cpp kdenetwork/kopete/libkopete/ui/accountselector.cpp
--- kdenetwork.orig/kopete/libkopete/ui/accountselector.cpp	Thu Jun  3 21:22:49 2004
+++ kdenetwork/kopete/libkopete/ui/accountselector.cpp	Mon Oct 18 13:26:04 2004
@@ -85,6 +85,7 @@
 AccountSelector::~AccountSelector()
 {
 	kdDebug(14010) << k_funcinfo << endl;
+	delete d;
 }
 
 
diff -urN -x CVS kdenetwork.orig/kopete/plugins/history/kopete_history_config.desktop kdenetwork/kopete/plugins/history/kopete_history_config.desktop
--- kdenetwork.orig/kopete/plugins/history/kopete_history_config.desktop	Tue Aug 24 08:31:09 2004
+++ kdenetwork/kopete/plugins/history/kopete_history_config.desktop	Mon Oct  4 15:25:49 2004
@@ -95,7 +95,7 @@
 Comment[sk]=Modul histórie
 Comment[sl]=Vstavek zgodovine
 Comment[sr]=Прикључак за историјат
-Comment[sr@Latn]=Dodatak za istorijat
+Comment[sr@Latn]=Priključak za istorijat
 Comment[sv]=Historikinsticksprogram
 Comment[ta]=வரலாற்று செருகல்
 Comment[tg]=Модули Номнависи сӯҳбатҳо
diff -urN -x CVS kdenetwork.orig/kopete/plugins/latex/kopete_latex.desktop kdenetwork/kopete/plugins/latex/kopete_latex.desktop
--- kdenetwork.orig/kopete/plugins/latex/kopete_latex.desktop	Tue Aug 31 09:22:31 2004
+++ kdenetwork/kopete/plugins/latex/kopete_latex.desktop	Mon Oct  4 15:25:50 2004
@@ -38,7 +38,7 @@
 Comment[sk]=Zobrazovanie výrazov Latex v okne rozhovoru
 Comment[sl]=Izris formul LaTeX v oknu za klepet
 Comment[sr]=Приказ Latex формула у прозору за ћаскање
-Comment[sr@Latn]=Приказ Latex формула у прозору за ћаскање
+Comment[sr@Latn]=Prikaz Latex formula u prozoru za ćaskanje
 Comment[sv]=Visa Latex-formler i chattfönstret
 Comment[ta]= Render Latex formulas in the chatwindow
 Comment[tg]=Формулаҳои Render Latex дар тирезаи чат
diff -urN -x CVS kdenetwork.orig/kopete/plugins/msginfo/kopete_msginfo.desktop kdenetwork/kopete/plugins/msginfo/kopete_msginfo.desktop
--- kdenetwork.orig/kopete/plugins/msginfo/kopete_msginfo.desktop	Tue Aug 31 09:22:31 2004
+++ kdenetwork/kopete/plugins/msginfo/kopete_msginfo.desktop	Mon Oct  4 15:25:50 2004
@@ -59,7 +59,7 @@
 Comment[sk]=Modul vytvorený pre výukový text 'Vytváranie modulov Kopete'
 Comment[sl]=Dodatek za navodila o pisanju vstavkov za Kopete
 Comment[sr]=Прикључак написан за приручник „Писање Kopete прикључака“
-Comment[sr@Latn]=Dodatak napisan za priručnik „Pisanje Kopete dodataka“
+Comment[sr@Latn]=Priključak napisan za priručnik „Pisanje Kopete priključaka“
 Comment[sv]=Insticksprogram skrivet för handledningen "Att skriva insticksprogram för Kopete"
 Comment[ta]=செருகல் 'Writing Kopete Plugins' பயிற்சிக்காக எழுதப்பட்டுள்ளது
 Comment[tg]=Модули навиштан барои воситаҳои ёдгирии 'Модулҳои навиштании Kopete' 
diff -urN -x CVS kdenetwork.orig/kopete/protocols/groupwise/kopete_groupwise.desktop kdenetwork/kopete/protocols/groupwise/kopete_groupwise.desktop
--- kdenetwork.orig/kopete/protocols/groupwise/kopete_groupwise.desktop	Tue Aug 31 09:22:31 2004
+++ kdenetwork/kopete/protocols/groupwise/kopete_groupwise.desktop	Mon Oct  4 15:25:50 2004
@@ -32,7 +32,7 @@
 Comment[pt_BR]=Mensageiro do GroupWise da Novell
 Comment[sl]=Sporočilnik za Novell GroupWise
 Comment[sr]=Novell-ов гласник GroupWise
-Comment[sr@Latn]=Novell-ов гласник GroupWise
+Comment[sr@Latn]=Novell-ov glasnik GroupWise
 Comment[sv]=Novell GroupWise-meddelandeklient
 Comment[ta]=Novell GroupWise உடனடி தூதர்
 Comment[tg]=Пайёмбари Novell GroupWise 
diff -urN -x CVS kdenetwork.orig/kopete/protocols/irc/ui/channellist.cpp kdenetwork/kopete/protocols/irc/ui/channellist.cpp
--- kdenetwork.orig/kopete/protocols/irc/ui/channellist.cpp	Thu Jul  8 15:50:43 2004
+++ kdenetwork/kopete/protocols/irc/ui/channellist.cpp	Sat Oct 16 15:15:40 2004
@@ -103,7 +103,6 @@
 	QToolTip::add( channelSearch, i18n( "You may search for channels on the IRC server for a text string entered here." ) );
 	QWhatsThis::add( channelSearch, i18n( "You may search for channels on the IRC server for a text string entered here.  For instance, you may type 'linux' to find channels that have something to do with linux." ) );
 	mSearchButton->setText( i18n( "S&earch" ) );
-	mSearchButton->setAccel( QKeySequence( i18n( "Alt+E" ) ) );
 	QToolTip::add( mSearchButton, i18n( "Perform a channel search." ) );
 	QWhatsThis::add( mSearchButton, i18n( "Perform a channel search.  Please be patient, as this can be slow depending on the number of channels on the server." ) );
 	QToolTip::add( mChannelList, i18n( "Double click on a channel to select it." ) );
diff -urN -x CVS kdenetwork.orig/kopete/protocols/msn/config/kopete_msn_config.desktop kdenetwork/kopete/protocols/msn/config/kopete_msn_config.desktop
--- kdenetwork.orig/kopete/protocols/msn/config/kopete_msn_config.desktop	Tue Aug 24 08:31:13 2004
+++ kdenetwork/kopete/protocols/msn/config/kopete_msn_config.desktop	Mon Oct  4 15:25:50 2004
@@ -44,7 +44,7 @@
 Name[sk]=Modul MSN
 Name[sl]=Vstavek za MSN
 Name[sr]=MSN прикључак
-Name[sr@Latn]=MSN dodatak
+Name[sr@Latn]=MSN priključak
 Name[sv]=MSN-insticksprogram
 Name[ta]=MSN செருகல்
 Name[tg]=Модули MSN
diff -urN -x CVS kdenetwork.orig/kopete/protocols/winpopup/config/kopete_wp_config.desktop kdenetwork/kopete/protocols/winpopup/config/kopete_wp_config.desktop
--- kdenetwork.orig/kopete/protocols/winpopup/config/kopete_wp_config.desktop	Tue Aug 24 08:31:15 2004
+++ kdenetwork/kopete/protocols/winpopup/config/kopete_wp_config.desktop	Mon Oct  4 15:25:51 2004
@@ -42,7 +42,7 @@
 Name[sk]=Modul WinPopup
 Name[sl]=Vstavek za WinPopup
 Name[sr]=WinPopup прикључак
-Name[sr@Latn]=WinPopup dodatak
+Name[sr@Latn]=WinPopup priključak
 Name[sv]=WinPopup-insticksprogram
 Name[ta]=WinPopup செருகல்
 Name[tg]=Модули WinPopup
diff -urN -x CVS kdenetwork.orig/kpf/kpfpropertiesdialogplugin.desktop kdenetwork/kpf/kpfpropertiesdialogplugin.desktop
--- kdenetwork.orig/kpf/kpfpropertiesdialogplugin.desktop	Tue Aug 31 09:22:34 2004
+++ kdenetwork/kpf/kpfpropertiesdialogplugin.desktop	Mon Oct  4 15:25:53 2004
@@ -23,7 +23,7 @@
 Name[sk]=Stránka vlastností priečinku KPF
 Name[sl]=Stran z lastnostmi imenika KPF
 Name[sr]=Страна са својствима KPF директоријума
-Name[sr@Latn]=Страна са својствима KPF директоријума
+Name[sr@Latn]=Strana sa svojstvima KPF direktorijuma
 Name[sv]=KPF-sida med katalogegenskaper
 Name[ta]=KPF directory properties Page
 Name[tg]=Саҳифаи Хусусиятҳои Феҳристи KPF
diff -urN -x CVS kdenetwork.orig/kppp/Kppp.desktop kdenetwork/kppp/Kppp.desktop
--- kdenetwork.orig/kppp/Kppp.desktop	Tue Aug 31 09:22:37 2004
+++ kdenetwork/kppp/Kppp.desktop	Mon Oct  4 15:25:56 2004
@@ -24,7 +24,7 @@
 GenericName[sk]=Pripojenie na internet cez modem
 GenericName[sl]=Orodje za klicni priklop na internet
 GenericName[sr]=Алат за повезивање на Интернет
-GenericName[sr@Latn]=Алат за повезивање на Интернет
+GenericName[sr@Latn]=Alat za povezivanje na Internet
 GenericName[sv]=Uppringningsverktyg för Internet
 GenericName[ta]=இணைய அழைப்புக் கருவி
 GenericName[tg]=Асбоб барои Зангзанӣ ба Интернет
diff -urN -x CVS kdenetwork.orig/kppp/general.h kdenetwork/kppp/general.h
--- kdenetwork.orig/kppp/general.h	Wed May 19 20:20:12 2004
+++ kdenetwork/kppp/general.h	Thu Oct 14 15:14:53 2004
@@ -2,7 +2,7 @@
  *
  *            kPPP: A pppd front end for the KDE project
  *
- * $Id$
+ * $Id$
  *
  *            Copyright (C) 1997 Bernd Johannes Wuebben
  *                   wuebben@math.cornell.edu
@@ -61,6 +61,7 @@
 public:
   ModemWidget(QWidget *parent=0, bool isnewmodem=true, const char *name=0);
 	bool save();
+	QLineEdit *connectName() { return connectname_l;}
 
 private slots:
 
diff -urN -x CVS kdenetwork.orig/kppp/modems.cpp kdenetwork/kppp/modems.cpp
--- kdenetwork.orig/kppp/modems.cpp	Sun Jul 11 08:09:24 2004
+++ kdenetwork/kppp/modems.cpp	Thu Oct 14 15:12:13 2004
@@ -219,10 +219,11 @@
     tabWindow->setCaption(tit);
     isnewmodem = false;
   }
-
   modem1 = new ModemWidget(tabWindow->addPage( i18n("&Device"), i18n("Serial Device")), isnewmodem );
   modem2 = new ModemWidget2(tabWindow->addPage( i18n("&Modem"), i18n("Modem Settings")));
+  connect ( modem1->connectName(), SIGNAL(textChanged ( const QString & )), this, SLOT(modemNameChanged(const QString & )));
 
+  modemNameChanged(modem1->connectName()->text());
   int result = 0;
   bool ok = false;
   while (!ok){
@@ -245,6 +246,10 @@
  return result;
 }
 
+void ModemsWidget::modemNameChanged(const QString & text)
+{
+  tabWindow->enableButtonOK( !text.isEmpty() );
+}
 
 QString ModemsWidget::prettyPrintVolume(unsigned int n) {
   int idx = 0;
diff -urN -x CVS kdenetwork.orig/kppp/modems.h kdenetwork/kppp/modems.h
--- kdenetwork.orig/kppp/modems.h	Fri May 14 18:57:06 2004
+++ kdenetwork/kppp/modems.h	Thu Oct 14 15:12:13 2004
@@ -55,6 +55,7 @@
   void newmodem();
   void deletemodem();
   void slotListBoxSelect(int);
+  void modemNameChanged(const QString &);
 
 private:
   int doTab();
diff -urN -x CVS kdenetwork.orig/krfb/kcm_krfb/kcmkrfb.desktop kdenetwork/krfb/kcm_krfb/kcmkrfb.desktop
--- kdenetwork.orig/krfb/kcm_krfb/kcmkrfb.desktop	Tue Aug 31 09:22:40 2004
+++ kdenetwork/krfb/kcm_krfb/kcmkrfb.desktop	Sun Oct  3 15:37:44 2004
@@ -47,7 +47,7 @@
 Name[ro]=Partajare ecran
 Name[ru]=Общий рабочий стол
 Name[sk]=Zdieľanie pracovnej plochy
-Name[sl]=Deljenje namizja
+Name[sl]=Souporaba namizja
 Name[sr]=Дељење радне површине
 Name[sr@Latn]=Deljenje radne površine
 Name[sv]=Dela ut skrivbordet
diff -urN -x CVS kdenetwork.orig/krfb/kinetd/eventsrc kdenetwork/krfb/kinetd/eventsrc
--- kdenetwork.orig/krfb/kinetd/eventsrc	Tue Aug 31 09:22:40 2004
+++ kdenetwork/krfb/kinetd/eventsrc	Sun Oct  3 15:37:45 2004
@@ -96,7 +96,6 @@
 Comment[sr]=Примљена је долазећа веза
 Comment[sr@Latn]=Primljena je dolazeća veza
 Comment[sv]=Tar emot inkommande anslutning
-Comment[ta]== உள்வரும் இணைப்பு ஏற்கப்பட்டது
 Comment[tg]=Пайвастшавии воридшаванда қабул гардид
 Comment[th]=ได้รับการเชื่อมต่อเข้ามา
 Comment[tr]=Gelen bağlantı alınıyor
diff -urN -x CVS kdenetwork.orig/krfb/krfb/krfb.desktop kdenetwork/krfb/krfb/krfb.desktop
--- kdenetwork.orig/krfb/krfb/krfb.desktop	Tue Aug 31 09:22:41 2004
+++ kdenetwork/krfb/krfb/krfb.desktop	Sun Oct  3 15:37:45 2004
@@ -43,7 +43,7 @@
 Name[ro]=Partajare ecran
 Name[ru]=Общий рабочий стол
 Name[sk]=Zdieľanie pracovnej plochy
-Name[sl]=Deljenje namizja
+Name[sl]=Souporaba namizja
 Name[sr]=Дељење радне површине
 Name[sr@Latn]=Deljenje radne površine
 Name[sv]=Dela ut skrivbordet
diff -urN -x CVS kdenetwork.orig/ksirc/KSOpenkSirc/open_ksirc.cpp kdenetwork/ksirc/KSOpenkSirc/open_ksirc.cpp
--- kdenetwork.orig/ksirc/KSOpenkSirc/open_ksirc.cpp	Sat Dec 13 04:19:08 2003
+++ kdenetwork/ksirc/KSOpenkSirc/open_ksirc.cpp	Mon Oct 18 14:19:12 2004
@@ -228,10 +228,11 @@
 {
   QCString utf8 = password.utf8();
   // Can contain NULL bytes after XORing
-  QByteArray result(utf8.length() << 1);
-  memcpy(result.data(), kapp->randomString(utf8.length()).latin1(), utf8.length());
-  for (unsigned int i = 0; i < utf8.length(); ++i)
-    result[i + utf8.length()] = utf8[i] ^ result[i];
+  unsigned int utf8Length(utf8.length());
+  QByteArray result(utf8Length << 1);
+  memcpy(result.data(), kapp->randomString(utf8Length).latin1(), utf8Length);
+  for (unsigned int i = 0; i < utf8Length; ++i)
+    result[i + utf8Length] = utf8[i] ^ result[i];
   return QString::fromLatin1(KCodecs::base64Encode(result));
 }
 
diff -urN -x CVS kdenetwork.orig/lanbrowsing/kio_lan/lan.desktop kdenetwork/lanbrowsing/kio_lan/lan.desktop
--- kdenetwork.orig/lanbrowsing/kio_lan/lan.desktop	Tue Aug 24 08:31:37 2004
+++ kdenetwork/lanbrowsing/kio_lan/lan.desktop	Tue Oct  5 15:20:41 2004
@@ -10,6 +10,7 @@
 Name[az]=Yerli Şəbəkə
 Name[be]=Лякальная сетка
 Name[bg]=Локална мрежа
+Name[br]=Rouedad lec'hel
 Name[bs]=Lokalna mreža
 Name[ca]=Xarxa local
 Name[cs]=Lokální síť
diff -urN -x CVS kdenetwork.orig/lanbrowsing/kio_lan/lisa.desktop kdenetwork/lanbrowsing/kio_lan/lisa.desktop
--- kdenetwork.orig/lanbrowsing/kio_lan/lisa.desktop	Tue Aug 31 09:22:45 2004
+++ kdenetwork/lanbrowsing/kio_lan/lisa.desktop	Mon Oct  4 15:25:58 2004
@@ -29,7 +29,7 @@
 Name[sk]=Prehliadač siete
 Name[sl]=Brskalnik po lokalnem omrežju
 Name[sr]=Претраживач LAN-а
-Name[sr@Latn]=Претраживач LAN-а
+Name[sr@Latn]=Pretraživač LAN-a
 Name[sv]=Bläddrare i lokalt nätverk
 Name[ta]=LAN உலாவி
 Name[tg]=Барраси Шабакаи Маҳаллӣ
diff -urN -x CVS kdenetwork.orig/wifi/kcmwifi/kcmwifi.desktop kdenetwork/wifi/kcmwifi/kcmwifi.desktop
--- kdenetwork.orig/wifi/kcmwifi/kcmwifi.desktop	Tue Aug 24 08:31:39 2004
+++ kdenetwork/wifi/kcmwifi/kcmwifi.desktop	Mon Oct  4 15:26:00 2004
@@ -116,7 +116,7 @@
 Keywords[pt_BR]=rede,orinoco,sem fio,wlan,wlan-ng
 Keywords[sl]=omrežje,orinoco,brezžično,wlan,wlan-ng
 Keywords[sr]=network,orinoco,wireless,wlan,wlan-ng,мрежа,ориноко,бежична,бежично
-Keywords[sr@Latn]=network,orinoco,wireless,wlan,wlan-ng,mreža,orinoko,bežična
+Keywords[sr@Latn]=network,orinoco,wireless,wlan,wlan-ng,mreža,orinoko,bežična,bežično
 Keywords[sv]=nätverk,orinoco,trådlöst,wlan,wlan-ng
 Keywords[tr]=ağ,orinoco,telsiz,wlan,wlan-ng
 Keywords[uk]=мережа,orinoco,радіо,бездротовий,зв'язок,wlan,wlan-ng
