--- kdenetwork-3.0.3/knode/knconfig.cpp--	2002-08-31 08:28:01.000000000 +0200
+++ kdenetwork-3.0.3/knode/knconfig.cpp	2002-08-31 08:30:29.000000000 +0200
@@ -80,6 +80,7 @@ void KNConfig::Identity::saveConfig(KCon
   c->writeEntry("UseSigGenerator",u_seSigGenerator);
   c->writeEntry("sigFile", s_igPath);
   c->writeEntry("sigText", s_igText);
+  c->sync();
 }
 
 
@@ -307,6 +308,8 @@ void KNConfig::Appearance::save()
   c->writeEntry("composerFont", f_onts[composer]);
   c->writeEntry("groupListFont", f_onts[groupList]);
   c->writeEntry("articleListFont", f_onts[articleList]);
+  c->sync();
+
 
   d_irty = false;
 }
@@ -626,6 +629,7 @@ void KNConfig::ReadNewsGeneral::save()
   conf->setGroup("CACHE");
   conf->writeEntry("collMemSize", c_ollCacheSize);
   conf->writeEntry("artMemSize", a_rtCacheSize);
+  conf->sync();
 
   d_irty = false;
 }
@@ -669,6 +673,7 @@ void KNConfig::ReadNewsNavigation::save(
   conf->writeEntry("markThreadReadCloseThread", m_arkThreadReadCloseThread);
   conf->writeEntry("ignoreThreadGoNext", i_gnoreThreadGoNext);
   conf->writeEntry("ignoreThreadCloseThread", i_gnoreThreadCloseThread);
+  conf->sync();
 
   d_irty = false;
 }
@@ -747,7 +752,7 @@ void KNConfig::ReadNewsViewer::save()
                   break;
   }
   conf->writeEntry("BrowserCommand", b_rowserCommand);
-
+  conf->sync();
   d_irty = false;
 }
 
@@ -836,7 +841,7 @@ void KNConfig::DisplayedHeaders::save()
     }
     headerConf.writeEntry("Flags",flags);
   }
-
+  headerConf.sync();
   d_irty = false;
 }
 
@@ -910,7 +915,7 @@ void KNConfig::Scoring::save()
 
   conf->writeEntry("ignoredThreshold", i_gnoredThreshold);
   conf->writeEntry("watchedThreshold", w_atchedThreshold);
-
+  conf->sync();
   d_irty = false;
 }
 
@@ -1024,7 +1029,7 @@ void KNConfig::PostNewsTechnical::save()
     else
       KNHelper::displayInternalFileError();
   }
-
+  conf->sync();
   d_irty = false;
 }
 
@@ -1142,7 +1147,7 @@ void KNConfig::PostNewsComposer::save()
   conf->writeEntry("useExternalEditor", u_seExtEditor);
   conf->writeEntry("Intro", i_ntro);
   conf->writeEntry("externalEditor", e_xternalEditor);
-
+  conf->sync();
   d_irty = false;
 }
 
@@ -1193,7 +1198,7 @@ void KNConfig::Cleanup::save()
   conf->writeEntry("readDays", r_eadMaxAge);
   conf->writeEntry("unreadDays", u_nreadMaxAge);
   conf->writeEntry("comInterval", c_ompactInterval);
-
+  conf->sync();
   d_irty = false;
 }
 
