Index: document.cpp
===================================================================
RCS file: /home/kde/kdenetwork/dcoprss/document.cpp,v
retrieving revision 1.12
diff -u -u -r1.12 document.cpp
--- kdenetwork/dcoprss/document.cpp~	18 Jan 2004 11:34:58 -0000	1.12
+++ kdenetwork/dcoprss/document.cpp	28 Jan 2004 17:09:18 -0000
@@ -76,7 +76,7 @@
 			connect(img, SIGNAL(gotPixmap(const QPixmap &)),
 				SLOT(pixmapLoaded(const QPixmap &)));
 			img->getPixmap();
-			pixmapUpdateing(DCOPRef(this));
+			pixmapUpdating(DCOPRef(this));
 		}
 		m_isLoading = false;
 		documentUpdated(DCOPRef(this));
@@ -246,7 +246,7 @@
 		Loader *loader = Loader::create(this,
 			SLOT(loadingComplete(Loader *, Document, Status)));
 		loader->loadFrom(KURL( m_Url ),  new FileRetriever());
-		documentUpdateing(DCOPRef(this));
+		documentUpdating(DCOPRef(this));
 	}
 	else
 	{
@@ -263,7 +263,7 @@
 				connect(img, SIGNAL(gotPixmap(const QPixmap &)),
 					SLOT(pixmapLoaded(const QPixmap &)));
 				img->getPixmap();
-				pixmapUpdateing(DCOPRef(this));
+				pixmapUpdating(DCOPRef(this));
 			}
 		}
 		*/
Index: service.h
===================================================================
RCS file: /home/kde/kdenetwork/dcoprss/service.h,v
retrieving revision 1.12
diff -u -u -r1.12 service.h
--- kdenetwork/dcoprss/service.h~	18 Jun 2003 12:27:52 -0000	1.12
+++ kdenetwork/dcoprss/service.h	28 Jan 2004 17:09:18 -0000
@@ -132,7 +132,7 @@
 		/**
 		* The pixmap is currently loading
 		**/
-		void pixmapUpdateing(DCOPRef);
+		void pixmapUpdating(DCOPRef);
 		/**
 		*  The pixmap is ready for viewing
 		*  you can then use dcopref->call("pixmap()"); to return it.
@@ -140,9 +140,9 @@
 		**/
 		void pixmapUpdated(DCOPRef);
 		/**
-		* The document is currently updateing
+		* The document is currently updating
 		**/
-		void documentUpdateing(DCOPRef);
+		void documentUpdating(DCOPRef);
 		/**
 		* The document is ready for viewing
 		* you can then use dcopref->call() to access its data
