diff -Nur kdenetwork-031204.orig/kopete/protocols/gadu/gaduprotocol.cpp kdenetwork-031204.chng/kopete/protocols/gadu/gaduprotocol.cpp
--- kdenetwork-031204.orig/kopete/protocols/gadu/gaduprotocol.cpp	2003-11-03 20:10:31.000000000 +0100
+++ kdenetwork-031204.chng/kopete/protocols/gadu/gaduprotocol.cpp	2003-12-12 19:48:13.622053008 +0100
@@ -49,8 +49,8 @@
 				"gg_ignored", "", i18n( "Blocked" ) ),
 			gaduStatusOffline_( KopeteOnlineStatus::Offline, GG_STATUS_NOT_AVAIL, this, GG_STATUS_NOT_AVAIL,
 				"gg_offline", i18n( "Go O&ffline" ), i18n( "Offline" ) ),
-			gaduStatusOfflineDescr_( KopeteOnlineStatus::Away, GG_STATUS_NOT_AVAIL_DESCR, this, GG_STATUS_NOT_AVAIL_DESCR,
-				"gg_offline_d", i18n( "Go A&way" ), i18n( "Offline" ) ),
+			gaduStatusOfflineDescr_( KopeteOnlineStatus::Offline, GG_STATUS_NOT_AVAIL_DESCR, this, GG_STATUS_NOT_AVAIL_DESCR,
+				"gg_offline_d", i18n( "Go O&ffline" ), i18n( "Offline" ) ),
 			gaduStatusBusy_(KopeteOnlineStatus::Away, GG_STATUS_BUSY, this, GG_STATUS_BUSY,
 				"gg_busy", i18n( "Go B&usy" ), i18n( "Busy" ) ),
 			gaduStatusBusyDescr_(KopeteOnlineStatus::Away, GG_STATUS_BUSY_DESCR, this, GG_STATUS_BUSY_DESCR,
