--- branches/KDE/3.5/kdenetwork/kopete/protocols/oscar/aim/aimcontact.cpp	2007/06/08 19:44:15	673026
+++ branches/KDE/3.5/kdenetwork/kopete/protocols/oscar/aim/aimcontact.cpp	2007/06/08 19:44:48	673027
@@ -440,6 +440,9 @@
 
 	s.replace ( QRegExp ( QString::fromLatin1("<br[ /]*>")), QString::fromLatin1("<br>") );
 
+	// strip left over line break
+	s.remove( QRegExp( QString::fromLatin1( "<br>$" ) ) );
+
 	kdDebug(14190) << k_funcinfo << "sending "
 		<< s << endl;
 
