Index: kopete/libkopete/avdevice/videodevice.cpp
===================================================================
--- kopete/libkopete/avdevice/videodevice.cpp	(.../tags/KDE/3.5.9/kdenetwork)	(wersja 780364)
+++ kopete/libkopete/avdevice/videodevice.cpp	(.../branches/KDE/3.5/kdenetwork)	(wersja 780364)
@@ -1965,7 +1965,7 @@
 
 // Packed RGB formats
 				case PIXELFORMAT_RGB332	: return V4L2_PIX_FMT_RGB332;	break;
-#if defined( V4L2_PIX_FMT_RGB44 )
+#if defined( V4L2_PIX_FMT_RGB444 )
 				case PIXELFORMAT_RGB444	: return V4L2_PIX_FMT_RGB444;	break;
 #endif
 				case PIXELFORMAT_RGB555	: return V4L2_PIX_FMT_RGB555;	break;

Zmiany atrybutów dla: .
___________________________________________________________________
Nazwa: svn:mergeinfo
   - 
Nazwa: svn:externals
   + admin https://svn.kde.org/home/kde/branches/KDE/3.5/kde-common/admin


