--- kdenetwork-3.0.98/kget/transfer.cpp.orig	Mon Nov  4 17:01:37 2002
+++ kdenetwork-3.0.98/kget/transfer.cpp	Fri Nov  8 11:40:23 2002
@@ -121,11 +121,11 @@
     connect(this, SIGNAL(log(uint, const QString &, const QString &)), kmain->logwin(), SLOT(logTransfer(uint, const QString &, const QString &)));
 
     // setup actions
-    m_paResume = new KAction(i18n("&Resume"), QIconSet(QPixmap(locate("appdata", "pics/tool_resume.png"))), 0, this, SLOT(slotResume()), this, "resume");
+    m_paResume = new KAction(i18n("&Resume"), "tool_resume.png", 0, this, SLOT(slotResume()), this, "resume");
 
-    m_paPause = new KAction(i18n("&Pause"), QIconSet(QPixmap(locate("appdata", "pics/tool_pause.png"))), 0, this, SLOT(slotRequestPause()), this, "pause");
+    m_paPause = new KAction(i18n("&Pause"), "tool_pause.png", 0, this, SLOT(slotRequestPause()), this, "pause");
 
-    m_paDelete = new KAction(i18n("&Delete"), QIconSet(QPixmap(locate("appdata", "pics/tool_delete.png"))), 0, this, SLOT(slotRequestRemove()), this, "delete");
+    m_paDelete = new KAction(i18n("&Delete"), "tool_delete.png", 0, this, SLOT(slotRequestRemove()), this, "delete");
 
     m_paRestart = new KAction(i18n("Re&start"), "tool_restart", 0, this, SLOT(slotRequestRestart()), this, "restart");
 
