--- kdenetwork-3.2.0/kopete/kopete/systemtray.cpp.orig	2004-01-28 20:17:30.393894968 +0100
+++ kdenetwork-3.2.0/kopete/kopete/systemtray.cpp	2004-01-28 20:18:57.465658048 +0100
@@ -140,24 +140,24 @@
 
 void KopeteSystemTray::startBlink( const QMovie &movie )
 {
-	kdDebug( 14010 ) << k_funcinfo << "starting movie." << endl;
+/*	kdDebug( 14010 ) << k_funcinfo << "starting movie." << endl;
 	const_cast<QMovie &>( movie ).unpause();
 	setMovie( movie );
-	mIsBlinking = true;
+	mIsBlinking = true;*/
 }
 
 void KopeteSystemTray::startBlink()
 {
-	if ( mMovie.isNull() )
+/*	if ( mMovie.isNull() )
 	{
-#if KDE_IS_VERSION(3, 1, 90)
+if KDE_IS_VERSION(3, 1, 90)
 		mMovie = KGlobal::iconLoader()->loadMovie( QString::fromLatin1( "newmessage" ), KIcon::Panel );
-#else
+else
 		mMovie = KopeteCompat::loadMovie( QString::fromLatin1( "newmessage" ), KIcon::Panel );
-#endif
+endif
 	}
 
-	startBlink( mMovie );
+	startBlink( mMovie );*/
 }
 
 void KopeteSystemTray::stopBlink()
@@ -167,8 +167,8 @@
 	else if ( mBlinkTimer->isActive() )
 		mBlinkTimer->stop();
 
-	if ( !mMovie.isNull() )
-		mMovie.pause();
+/*	if ( !mMovie.isNull() )
+		mMovie.pause();*/
 
 	mIsBlinkIcon = false;
 	mIsBlinking = false;
--- kdenetwork-3.2.0/kopete/libkopete/compat/loadmovie.cpp.orig	2004-01-28 20:16:10.769999640 +0100
+++ kdenetwork-3.2.0/kopete/libkopete/compat/loadmovie.cpp	2004-01-28 20:29:06.957001280 +0100
@@ -40,15 +40,13 @@
 // KIconLoader::loadMovie broken in KDE <= 3.1 if current icon set is not crystal
 QMovie KopeteCompat::loadMovie( const QString &name, KIcon::Group group )
 {
-	QString file = name + ".mng";
+/*	QString file = name + ".mng";
 	QString appname = KGlobal::instance()->instanceName();
 	KIconTheme crystal( QString::fromLatin1("crystalsvg"), appname );
 
 	KIcon icon = crystal.iconPath( file, iconSize( group ), KIcon::MatchExact );
 	if( !icon.isValid() )
 		icon = crystal.iconPath( file, iconSize( group ), KIcon::MatchBest );
-	if( !icon.isValid() )
-		return QMovie();
-
-	return QMovie( icon.path );
+	if( !icon.isValid() )*/
+		return QMovie;
 }
--- kdenetwork-3.2.0/kopete/kopete/kopetewindow.cpp.orig	2004-01-28 20:19:33.636159296 +0100
+++ kdenetwork-3.2.0/kopete/kopete/kopetewindow.cpp	2004-01-28 20:21:28.387714408 +0100
@@ -574,16 +574,16 @@
 
 	// Because we want null pixmaps to detect the need for a loadMovie
 	// we can't use the SmallIcon() method directly
-	KIconLoader *loader = KGlobal::instance()->iconLoader();
+/*	KIconLoader *loader = KGlobal::instance()->iconLoader();
 
-#if KDE_IS_VERSION(3, 1, 90)
+if KDE_IS_VERSION(3, 1, 90)
 	QMovie mv = loader->loadMovie( status.overlayIcon(),
-#else
+else
 	QMovie mv = KopeteCompat::loadMovie( status.overlayIcon(),
-#endif
-		 KIcon::Small);
+endif
+		 KIcon::Small);*/
 
-	if ( mv.isNull() )
+	if ( 1 )
 	{
 		// No movie found, fallback to pixmap
 		// Get the icon for our status
@@ -619,7 +619,7 @@
 	else
 	{
 		//kdDebug( 14000 ) << k_funcinfo << "Using movie."  << endl;
-		i->setMovie( mv );
+//		i->setMovie( mv );
 	}
 }
 
--- kdenetwork-3.2.0/kopete/kopete/chatwindow/kopeteemailwindow.cpp.orig	2004-01-28 20:22:21.951571464 +0100
+++ kdenetwork-3.2.0/kopete/kopete/chatwindow/kopeteemailwindow.cpp	2004-01-28 20:26:10.148880200 +0100
@@ -276,18 +276,18 @@
 
 	// The animated toolbarbutton
 	d->normalIcon = QPixmap( BarIcon( QString::fromLatin1( "kopete" ) ) );
-#if KDE_IS_VERSION(3, 1, 90)
+/*if KDE_IS_VERSION(3, 1, 90)
 	d->animIcon = KGlobal::iconLoader()->loadMovie( QString::fromLatin1( "newmessage" ), KIcon::Toolbar);
-#else
+else
 	d->animIcon = KopeteCompat::loadMovie( QString::fromLatin1( "newmessage" ), KIcon::Toolbar);
-#endif
+endif
 	d->animIcon.pause();
 
 	d->anim = new QLabel( this, "kde toolbar widget" );
 	d->anim->setMargin(5);
 	d->anim->setPixmap( d->normalIcon );
 	//toolBar()->insertWidget( 99, d->anim->width(), d->anim );
-	new KWidgetAction( d->anim , i18n("Toolbar Animation") , 0, 0 , 0 , coll , "toolbar_animation");
+	new KWidgetAction( d->anim , i18n("Toolbar Animation") , 0, 0 , 0 , coll , "toolbar_animation");*/
 
 	setXMLFile( QString::fromLatin1( "kopetechatwindow.rc" ) );
 	createGUI( d->editpart );
@@ -518,8 +518,8 @@
 void KopeteEmailWindow::sendMessage()
 {
 	d->sendInProgress = true;
-	d->anim->setMovie( d->animIcon );
-	d->animIcon.unpause();
+//	d->anim->setMovie( d->animIcon );
+//	d->animIcon.unpause();
 	d->btnReplySend->setEnabled( false );
 	d->txtEntry->setEnabled( false );
 	KopeteMessage sentMessage = currentMessage();
