--- kdenetwork/krfb/krfb/krfb.desktop~	2003-09-06 13:31:55.000000000 +0200
+++ kdenetwork/krfb/krfb/krfb.desktop	2003-09-06 13:32:42.000000000 +0200
@@ -51,4 +51,4 @@
 Name[zh_CN]=桌面共享
 Name[zh_TW]=桌面分享
 Name[zu]=Ukwahlulelana kwe-Desktop
-Categories=Qt;KDE;System
+Categories=Qt;KDE;Network
--- kdenetwork-030914/kopete/kopete/kopete.desktop~	2003-09-16 19:40:38.000000000 +0200
+++ kdenetwork-030914/kopete/kopete/kopete.desktop	2003-09-17 09:46:17.000000000 +0200
@@ -10,4 +10,4 @@
 Type=Application
 Terminal=0
 Icon=kopete
-Categories=Qt;KDE;Network
+Categories=Qt;KDE;Network;X-Communication
