--- kdenetwork-3.0.8/kmail/kmmainwin.cpp.orig	Sun Sep 22 07:51:33 2002
+++ kdenetwork-3.0.8/kmail/kmmainwin.cpp	Mon Sep 30 04:20:58 2002
@@ -2267,7 +2267,7 @@
   (void) new KAction( i18n("&Find in Message..."), "find", KStdAccel::shortcut(KStdAccel::Find), this,
 		      SLOT(slotFind()), actionCollection(), "find_in_messages" );
 
-  (void) new KAction( i18n("Select &All Messages"), Key_K, this,
+  (void) new KAction( i18n("Select &All Messages"), "ok", Key_K, this,
 		      SLOT(slotMarkAll()), actionCollection(), "mark_all_messages" );
 
   (void) new KAction( i18n("Select Message &Text"), KStdAccel::shortcut(KStdAccel::SelectAll), this,
@@ -2296,10 +2296,10 @@
   removeFolderAction = new KAction( i18n("&Delete Folder"), "editdelete", 0, this,
 		      SLOT(slotRemoveFolder()), actionCollection(), "delete_folder" );
 
-  preferHtmlAction = new KToggleAction( i18n("Prefer &HTML to Plain Text"), 0, this,
+  preferHtmlAction = new KToggleAction( i18n("Prefer &HTML to Plain Text"), "html", 0, this,
 		      SLOT(slotOverrideHtml()), actionCollection(), "prefer_html" );
 
-  threadMessagesAction = new KToggleAction( i18n("&Thread Messages"), 0, this,
+  threadMessagesAction = new KToggleAction( i18n("&Thread Messages"), "view_tree", 0, this,
 		      SLOT(slotOverrideThread()), actionCollection(), "thread_messages" );
 
   //----- Message Menu
@@ -2625,11 +2625,11 @@
 		     SLOT(slotCollapseAllThreads()),
 		     actionCollection(), "collapse_all_threads" );
 
-  toggleFixFontAction = new KToggleAction( i18n("Use Fi&xed Font"),
+  toggleFixFontAction = new KToggleAction( i18n("Use Fi&xed Font"), "text",
 			Key_X, this, SLOT(slotToggleFixedFont()),
 			actionCollection(), "toggle_fixedfont" );
 
-  viewSourceAction = new KAction( i18n("&View Source"), Key_V, this,
+  viewSourceAction = new KAction( i18n("&View Source"), "source",  Key_V, this,
 		      SLOT(slotShowMsgSrc()), actionCollection(), "view_source" );
 
 
--- kdenetwork-3.0.8/kmail/kmcomposewin.cpp.orig	Sun Sep 22 07:51:33 2002
+++ kdenetwork-3.0.8/kmail/kmcomposewin.cpp	Mon Sep 30 04:14:26 2002
@@ -840,7 +840,7 @@
   (void) new KAction (i18n("Cl&ean Spaces"), 0, this, SLOT(slotCleanSpace()),
                       actionCollection(), "clean_spaces");
 
-  fixedFontAction = new KToggleAction( i18n("Use Fi&xed Font"), 0, this,
+  fixedFontAction = new KToggleAction( i18n("Use Fi&xed Font"), "text", 0, this,
                       SLOT(slotUpdateFont()), actionCollection(), "toggle_fixedfont" );
 
   //these are checkable!!!
