diff -x 'Makefile*' -x '*.*o*' -x '.*' -ur kopete.orig/kopete/chatwindow/kopetechatwindow.cpp kopete/kopete/chatwindow/kopetechatwindow.cpp
--- kopete.orig/kopete/chatwindow/kopetechatwindow.cpp	2004-01-17 13:54:09.000000000 +0100
+++ kopete/kopete/chatwindow/kopetechatwindow.cpp	2004-01-29 15:34:41.066609328 +0100
@@ -459,6 +459,7 @@
 
 	//The Sending movie
 	normalIcon = QPixmap( BarIcon( QString::fromLatin1( "kopete" ) ) );
+	/*
 #if KDE_IS_VERSION(3, 1, 90)
 	animIcon = KGlobal::iconLoader()->loadMovie( QString::fromLatin1( "newmessage" ), KIcon::Toolbar);
 #else
@@ -469,7 +470,7 @@
 	// showing it. This eats resources, and also triggers a pixmap leak in
 	// QMovie in at least Qt 3.1, Qt 3.2 and the current Qt 3.3 beta
 	animIcon.pause();
-
+*/
 	// we can't set the tool bar as parent, if we do, it will be deleted when we configure toolbars
 	anim = new QLabel( QString::null, 0L ,"kde toolbar widget" );
 	anim->setMargin(5);
@@ -882,8 +883,8 @@
 
 	if ( m_activeView->sendInProgress() )
 	{
-		anim->setMovie( animIcon );
-		animIcon.unpause();
+//		anim->setMovie( animIcon );
+//		animIcon.unpause();
 	}
 	else
 	{
@@ -946,8 +947,8 @@
 {
 	if ( m_activeView )
 	{
-		anim->setMovie( animIcon );
-		animIcon.unpause();
+/*		anim->setMovie( animIcon );
+		animIcon.unpause();*/
 		m_activeView->sendMessage();
 	}
 }
diff -x 'Makefile*' -x '*.*o*' -x '.*' -ur kopete.orig/kopete/chatwindow/kopeteemailwindow.cpp kopete/kopete/chatwindow/kopeteemailwindow.cpp
--- kopete.orig/kopete/chatwindow/kopeteemailwindow.cpp	2004-01-17 13:54:09.000000000 +0100
+++ kopete/kopete/chatwindow/kopeteemailwindow.cpp	2004-01-29 15:35:10.722101008 +0100
@@ -276,18 +276,18 @@
 
 	// The animated toolbarbutton
 	d->normalIcon = QPixmap( BarIcon( QString::fromLatin1( "kopete" ) ) );
-#if KDE_IS_VERSION(3, 1, 90)
+/*if KDE_IS_VERSION(3, 1, 90)
 	d->animIcon = KGlobal::iconLoader()->loadMovie( QString::fromLatin1( "newmessage" ), KIcon::Toolbar);
-#else
+else
 	d->animIcon = KopeteCompat::loadMovie( QString::fromLatin1( "newmessage" ), KIcon::Toolbar);
-#endif
+endif
 	d->animIcon.pause();
 
 	d->anim = new QLabel( this, "kde toolbar widget" );
 	d->anim->setMargin(5);
 	d->anim->setPixmap( d->normalIcon );
 	//toolBar()->insertWidget( 99, d->anim->width(), d->anim );
-	new KWidgetAction( d->anim , i18n("Toolbar Animation") , 0, 0 , 0 , coll , "toolbar_animation");
+	new KWidgetAction( d->anim , i18n("Toolbar Animation") , 0, 0 , 0 , coll , "toolbar_animation");*/
 
 	setXMLFile( QString::fromLatin1( "kopetechatwindow.rc" ) );
 	createGUI( d->editpart );
@@ -518,8 +518,8 @@
 void KopeteEmailWindow::sendMessage()
 {
 	d->sendInProgress = true;
-	d->anim->setMovie( d->animIcon );
-	d->animIcon.unpause();
+//	d->anim->setMovie( d->animIcon );
+//	d->animIcon.unpause();
 	d->btnReplySend->setEnabled( false );
 	d->txtEntry->setEnabled( false );
 	KopeteMessage sentMessage = currentMessage();
diff -x 'Makefile*' -x '*.*o*' -x '.*' -ur kopete.orig/kopete/kopetewindow.cpp kopete/kopete/kopetewindow.cpp
--- kopete.orig/kopete/kopetewindow.cpp	2004-01-17 13:54:09.000000000 +0100
+++ kopete/kopete/kopetewindow.cpp	2004-01-29 15:29:29.692945320 +0100
@@ -575,15 +575,15 @@
 	// Because we want null pixmaps to detect the need for a loadMovie
 	// we can't use the SmallIcon() method directly
 	KIconLoader *loader = KGlobal::instance()->iconLoader();
-
-#if KDE_IS_VERSION(3, 1, 90)
+/*
+if KDE_IS_VERSION(3, 1, 90)
 	QMovie mv = loader->loadMovie( status.overlayIcon(),
-#else
+else
 	QMovie mv = KopeteCompat::loadMovie( status.overlayIcon(),
-#endif
-		 KIcon::Small);
+endif
+		 KIcon::Small);*/
 
-	if ( mv.isNull() )
+	if ( 1 )
 	{
 		// No movie found, fallback to pixmap
 		// Get the icon for our status
@@ -619,7 +619,7 @@
 	else
 	{
 		//kdDebug( 14000 ) << k_funcinfo << "Using movie."  << endl;
-		i->setMovie( mv );
+//		i->setMovie( mv );
 	}
 }
 
diff -x 'Makefile*' -x '*.*o*' -x '.*' -ur kopete.orig/kopete/systemtray.cpp kopete/kopete/systemtray.cpp
--- kopete.orig/kopete/systemtray.cpp	2004-01-17 13:54:09.000000000 +0100
+++ kopete/kopete/systemtray.cpp	2004-01-29 15:28:10.225026280 +0100
@@ -36,7 +36,7 @@
 #include "kopeteaccount.h"
 
 #if !KDE_IS_VERSION(3, 1, 90)
-#include "loadmovie.h"
+#include "load`movie.h"
 #endif
 
 KopeteSystemTray* KopeteSystemTray::s_systemTray = 0L;
@@ -140,35 +140,35 @@
 
 void KopeteSystemTray::startBlink( const QMovie &movie )
 {
-	kdDebug( 14010 ) << k_funcinfo << "starting movie." << endl;
+/*	kdDebug( 14010 ) << k_funcinfo << "starting movie." << endl;
 	const_cast<QMovie &>( movie ).unpause();
 	setMovie( movie );
-	mIsBlinking = true;
+	mIsBlinking = true;*/
 }
 
 void KopeteSystemTray::startBlink()
 {
-	if ( mMovie.isNull() )
+/*	if ( mMovie.isNull() )
 	{
-#if KDE_IS_VERSION(3, 1, 90)
+if KDE_IS_VERSION(3, 1, 90)
 		mMovie = KGlobal::iconLoader()->loadMovie( QString::fromLatin1( "newmessage" ), KIcon::Panel );
-#else
+else
 		mMovie = KopeteCompat::loadMovie( QString::fromLatin1( "newmessage" ), KIcon::Panel );
-#endif
+endif
 	}
 
-	startBlink( mMovie );
+	startBlink( mMovie );*/
 }
 
 void KopeteSystemTray::stopBlink()
 {
-	if ( movie() )
-		kdDebug( 14010 ) << k_funcinfo << "stopping movie." << endl;
-	else if ( mBlinkTimer->isActive() )
+/*	if ( movie() )
+		kdDebug( 14010 ) << k_funcinfo << "stopping movie." << endl;*/
+	if ( mBlinkTimer->isActive() )
 		mBlinkTimer->stop();
 
-	if ( !mMovie.isNull() )
-		mMovie.pause();
+/*	if ( !mMovie.isNull() )
+		mMovie.pause();*/
 
 	mIsBlinkIcon = false;
 	mIsBlinking = false;
diff -x 'Makefile*' -x '*.*o*' -x '.*' -ur kopete.orig/libkopete/compat/loadmovie.cpp kopete/libkopete/compat/loadmovie.cpp
--- kopete.orig/libkopete/compat/loadmovie.cpp	2003-11-15 00:42:09.000000000 +0100
+++ kopete/libkopete/compat/loadmovie.cpp	2004-01-28 20:46:38.000000000 +0100
@@ -40,15 +40,13 @@
 // KIconLoader::loadMovie broken in KDE <= 3.1 if current icon set is not crystal
 QMovie KopeteCompat::loadMovie( const QString &name, KIcon::Group group )
 {
-	QString file = name + ".mng";
+/*	QString file = name + ".mng";
 	QString appname = KGlobal::instance()->instanceName();
 	KIconTheme crystal( QString::fromLatin1("crystalsvg"), appname );
 
 	KIcon icon = crystal.iconPath( file, iconSize( group ), KIcon::MatchExact );
 	if( !icon.isValid() )
 		icon = crystal.iconPath( file, iconSize( group ), KIcon::MatchBest );
-	if( !icon.isValid() )
-		return QMovie();
-
-	return QMovie( icon.path );
+	if( !icon.isValid() )*/
+		return QMovie;
 }
