Index: libwinpopup.cpp
===================================================================
--- kdenetwork-3.5.1/kopete/protocols/winpopup/libwinpopup/libwinpopup.cpp	(Revision 505983)
+++ kdenetwork-3.5.1/kopete/protocols/winpopup/libwinpopup/libwinpopup.cpp	(Arbeitskopie)
@@ -98,27 +98,26 @@
 {
 	QDir dir(WP_POPUP_DIR);
 	if (! dir.exists()) {
-		int tmpYesNo =  KMessageBox::warningYesNo(Kopete::UI::Global::mainWidget(), i18n("Working directory /var/lib/winpopup/ does not exist.\n"
+		int tmpYesNo =  KMessageBox::warningYesNo(Kopete::UI::Global::mainWidget(), i18n("Working directory " WP_POPUP_DIR " does not exist.\n"
 								     "If you have not configured anything yet (samba) it may be better to call\n"
 								     "Install Into Samba (Configure... -> Account -> Edit)\n"
 								     "Should the directory be created? (May need root password)"), QString::null, i18n("Create Directory"), i18n("Do Not Create"));
 		if (tmpYesNo == KMessageBox::Yes) {
-			QString kdesuArgs = "mkdir -p -m 0777 /var/lib/winpopup";
+			QString kdesuArgs = "mkdir -p -m 0777 "WP_POPUP_DIR;
 			if (KApplication::kdeinitExecWait("kdesu", kdesuArgs) == 0) return true;
 		}
 	} else {
-		KFileItem tmpFileItem = KFileItem(KFileItem::Unknown, KFileItem::Unknown, "/var/lib/winpopup/");
+		KFileItem tmpFileItem = KFileItem(KFileItem::Unknown, KFileItem::Unknown, WP_POPUP_DIR);
 		QString tmpPerms = tmpFileItem.permissionsString();
 
-		if (tmpPerms != "drwxrwxrwx") {
+		if (tmpPerms.startsWith("drwxrwxrwx") == false) {
 
-			kdDebug(14170) << "Perms not ok!" << endl;
+			kdDebug(14170) << WP_POPUP_DIR" Perms not ok!" << endl;
 
-			int tmpYesNo =  KMessageBox::warningYesNo(Kopete::UI::Global::mainWidget(), i18n("Permissions of the working directory "
-									     "/var/lib/winpopup/ are wrong!\n"
+			int tmpYesNo =  KMessageBox::warningYesNo(Kopete::UI::Global::mainWidget(), i18n("Permissions of the working directory " WP_POPUP_DIR " are wrong!\n"
 									     "Fix? (May need root password)"), QString::null, i18n("Fix"), i18n("Do Not Fix"));
 			if (tmpYesNo == KMessageBox::Yes) {
-				QString kdesuArgs = "chmod 0777 /var/lib/winpopup";
+				QString kdesuArgs = "chmod 0777 " WP_POPUP_DIR;
 				if (KApplication::kdeinitExecWait("kdesu", kdesuArgs) == 0) return true;
 			}
 		} else {
@@ -274,7 +273,7 @@
 											"Fix? (May need root password)"), QString::null, i18n("Fix"), i18n("Do Not Fix"));
 						if (tmpYesNo == KMessageBox::Yes) {
 							QFileInfo messageFileInfo(messageFile);
-							QString kdesuArgs = "chmod 0666 /var/lib/winpopup/" + messageFileInfo.fileName();
+							QString kdesuArgs = "chmod 0666 "WP_POPUP_DIR + messageFileInfo.fileName();
 							if (KApplication::kdeinitExecWait("kdesu", kdesuArgs) == 0) {
 								if (!messageFile.remove())
 									KMessageBox::error(Kopete::UI::Global::mainWidget(), i18n("Still cannot remove it; please fix manually."));
