Index: krfb/krfb/eventsrc
===================================================================
--- krfb/krfb/eventsrc	(.../tags/KDE/3.4.2/kdenetwork)	(revision 442272)
+++ krfb/krfb/eventsrc	(.../branches/KDE/3.4/kdenetwork)	(revision 442272)
@@ -568,7 +568,7 @@
 Comment[cy]=Anfonodd y person gwahodd cyfrinair annilys.  Gwrthodwyd y cysylltiad.
 Comment[da]=Den inviterede part sendte et ugyldigt kodeord. Forbindelse afslået.
 Comment[de]=Die eingeladene Partei hat ein ungültiges Passwort gesendet: Verbindung abgelehnt.
-Comment[el]=Η προσκαλεσμένη ομάδα έστειλε άκυρο κωδικό πρόσβασης. Η σύνδεση απορρίφθηκε.
+Comment[el]=Η προσκαλεσμένη ομάδα έστειλε μη έγκυρο κωδικό πρόσβασης. Η σύνδεση απορρίφθηκε.
 Comment[eo]=La invitita kliento sendis nevalidan pasvorton. Konekto rifuzita.
 Comment[es]=La parte invitada envió una contraseña incorrecta. Conexión rechazada.
 Comment[et]=Kutsutu saatis vigase parooli. Ühendusest keelduti.
Index: kopete/plugins/alias/kopete_alias.desktop
===================================================================
--- kopete/plugins/alias/kopete_alias.desktop	(.../tags/KDE/3.4.2/kdenetwork)	(revision 442272)
+++ kopete/plugins/alias/kopete_alias.desktop	(.../branches/KDE/3.4/kdenetwork)	(revision 442272)
@@ -18,6 +18,7 @@
 Name[ar]=اسم مستعار
 Name[be]=Псэўданім
 Name[bg]=Запазени думи
+Name[br]=Lesanv
 Name[ca]=Àlies
 Name[cy]=Ffugenw
 Name[el]=Αντιστοίχηση
Index: kopete/plugins/alias/kopete_alias_config.desktop
===================================================================
--- kopete/plugins/alias/kopete_alias_config.desktop	(.../tags/KDE/3.4.2/kdenetwork)	(revision 442272)
+++ kopete/plugins/alias/kopete_alias_config.desktop	(.../branches/KDE/3.4/kdenetwork)	(revision 442272)
@@ -14,6 +14,7 @@
 Name[ar]=اسم مستعار
 Name[be]=Псэўданім
 Name[bg]=Запазени думи
+Name[br]=Lesanv
 Name[ca]=Àlies
 Name[cy]=Ffugenw
 Name[el]=Αντιστοίχηση
Index: kopete/plugins/highlight/kopete_highlight.desktop
===================================================================
--- kopete/plugins/highlight/kopete_highlight.desktop	(.../tags/KDE/3.4.2/kdenetwork)	(revision 442272)
+++ kopete/plugins/highlight/kopete_highlight.desktop	(.../branches/KDE/3.4/kdenetwork)	(revision 442272)
@@ -18,6 +18,7 @@
 Name[ar]=تمييز
 Name[be]=Падсьветка
 Name[bg]=Открояване
+Name[br]=Splannadur :
 Name[bs]=Isticanje
 Name[ca]=Ressaltat
 Name[cs]=Zvýraznění
Index: kopete/plugins/highlight/kopete_highlight_config.desktop
===================================================================
--- kopete/plugins/highlight/kopete_highlight_config.desktop	(.../tags/KDE/3.4.2/kdenetwork)	(revision 442272)
+++ kopete/plugins/highlight/kopete_highlight_config.desktop	(.../branches/KDE/3.4/kdenetwork)	(revision 442272)
@@ -14,6 +14,7 @@
 Name[ar]=تمييز
 Name[be]=Падсьветка
 Name[bg]=Открояване
+Name[br]=Splannadur :
 Name[bs]=Isticanje
 Name[ca]=Ressaltat
 Name[cs]=Zvýraznění
Index: kopete/plugins/latex/kopete_latex_config.desktop
===================================================================
--- kopete/plugins/latex/kopete_latex_config.desktop	(.../tags/KDE/3.4.2/kdenetwork)	(revision 442272)
+++ kopete/plugins/latex/kopete_latex_config.desktop	(.../branches/KDE/3.4/kdenetwork)	(revision 442272)
@@ -14,6 +14,7 @@
 Name[ar]=تمييز
 Name[be]=Падсьветка
 Name[bg]=Открояване
+Name[br]=Splannadur :
 Name[bs]=Isticanje
 Name[ca]=Ressaltat
 Name[cs]=Zvýraznění
Index: kopete/protocols/gadu/libgadu/libgadu.h
===================================================================
--- kopete/protocols/gadu/libgadu/libgadu.h	(.../tags/KDE/3.4.2/kdenetwork)	(revision 442272)
+++ kopete/protocols/gadu/libgadu/libgadu.h	(.../branches/KDE/3.4/kdenetwork)	(revision 442272)
@@ -33,6 +33,10 @@
 extern "C" {
 #endif
 
+#ifdef HAVE_STDINT_H
+#include <stdint.h>
+#endif
+	
 #include <libgadu-config.h>
 #include <sys/types.h>
 #include <stdio.h>
Index: kopete/protocols/oscar/liboscar/ssimodifytask.cpp
===================================================================
--- kopete/protocols/oscar/liboscar/ssimodifytask.cpp	(.../tags/KDE/3.4.2/kdenetwork)	(revision 442272)
+++ kopete/protocols/oscar/liboscar/ssimodifytask.cpp	(.../branches/KDE/3.4/kdenetwork)	(revision 442272)
@@ -152,6 +152,8 @@
 
 bool SSIModifyTask::renameGroup( const QString& oldName, const QString & newName )
 {
+	return false; //rename group is unimplemented
+	
 	m_opType = Rename;
 	m_opSubject = Group;
 	if ( oldName == newName )
Index: kopete/kopete/eventsrc
===================================================================
--- kopete/kopete/eventsrc	(.../tags/KDE/3.4.2/kdenetwork)	(revision 442272)
+++ kopete/kopete/eventsrc	(.../branches/KDE/3.4/kdenetwork)	(revision 442272)
@@ -491,6 +491,7 @@
 Name[ar]=تمييز
 Name[be]=Падсьветка
 Name[bg]=Открояване
+Name[br]=Splannadur :
 Name[bs]=Isticanje
 Name[ca]=Ressaltat
 Name[cs]=Zvýraznění
Index: kopete/libkopete/kopetemessage.cpp
===================================================================
--- kopete/libkopete/kopetemessage.cpp	(.../tags/KDE/3.4.2/kdenetwork)	(revision 442272)
+++ kopete/libkopete/kopetemessage.cpp	(.../branches/KDE/3.4/kdenetwork)	(revision 442272)
@@ -674,6 +674,7 @@
 	return doc;
 }
 
+// KDE4: Move that to a utils class/namespace
 QString Message::decodeString( const QCString &message, const QTextCodec *providedCodec, bool *success )
 {
 	/*
@@ -694,7 +695,7 @@
 	charsToCheck = 128 > charsToCheck ? charsToCheck : 128;
 
 	//They are providing a possible codec. Check if it is valid
-	if( providedCodec && providedCodec->heuristicContentMatch( message, charsToCheck ) >= charsToCheck )
+	if( providedCodec && providedCodec->heuristicContentMatch( message, charsToCheck ) >= 0 )
 	{
 		//All chars decodable.
 		return providedCodec->toUnicode( message );
@@ -709,7 +710,7 @@
 
 	//Try codecForContent - exact match
 	QTextCodec *testCodec = QTextCodec::codecForContent(message, charsToCheck);
-	if( testCodec && testCodec->heuristicContentMatch( message, charsToCheck ) >= charsToCheck )
+	if( testCodec && testCodec->heuristicContentMatch( message, charsToCheck ) >= 0 )
 	{
 		//All chars decodable.
 		return testCodec->toUnicode( message );
@@ -723,7 +724,7 @@
 
 	//Try local codec
 	testCodec = QTextCodec::codecForLocale();
-	if( testCodec && testCodec->heuristicContentMatch( message, charsToCheck ) >= charsToCheck )
+	if( testCodec && testCodec->heuristicContentMatch( message, charsToCheck ) >= 0 )
 	{
 		//All chars decodable.
 		kdDebug(14000) << k_funcinfo << "Using locale's codec" << endl;
@@ -732,7 +733,7 @@
 
 	//Try latin1 codec
 	testCodec = QTextCodec::codecForMib(4);
-	if( testCodec && testCodec->heuristicContentMatch( message, charsToCheck ) >= charsToCheck )
+	if( testCodec && testCodec->heuristicContentMatch( message, charsToCheck ) >= 0 )
 	{
 		//All chars decodable.
 		kdDebug(14000) << k_funcinfo << "Using latin1" << endl;
Index: knewsticker/eventsrc
===================================================================
--- knewsticker/eventsrc	(.../tags/KDE/3.4.2/kdenetwork)	(revision 442272)
+++ knewsticker/eventsrc	(.../branches/KDE/3.4/kdenetwork)	(revision 442272)
@@ -198,7 +198,7 @@
 Name[cy]=Ffeil RDF annilys
 Name[da]=Ugyldig RDF-fil
 Name[de]=Ungültige RDF-Datei
-Name[el]=Άκυρο αρχείο RDF
+Name[el]=Μη έγκυρο αρχείο RDF
 Name[eo]=Nevalida RDF-dosierojn
 Name[es]=Archivo RDF no válido
 Name[et]=Vigane RDF-fail
Index: kget/dlgIndividual.cpp
===================================================================
--- kget/dlgIndividual.cpp	(.../tags/KDE/3.4.2/kdenetwork)	(revision 442272)
+++ kget/dlgIndividual.cpp	(.../branches/KDE/3.4/kdenetwork)	(revision 442272)
@@ -196,7 +196,7 @@
     advanced = ksettings.b_advancedIndividual;
     slotToggleAdvanced();
 
-    resize( sizeHint() );
+    resize( minimumSizeHint() );
 
     //bool keepOpenChecked = false;
     //bool noCaptionYet = true;
Index: Makefile.cvs
===================================================================
--- Makefile.cvs	(.../tags/KDE/3.4.2/kdenetwork)	(revision 0)
+++ Makefile.cvs	(.../branches/KDE/3.4/kdenetwork)	(revision 442272)
@@ -0,0 +1,16 @@
+
+all: 
+	@echo "This Makefile is only for the CVS repository"
+	@echo "This will be deleted before making the distribution"
+	@echo ""
+	@if test ! -d admin; then \
+	   echo "Please recheckout this module!" ;\
+	   echo "for cvs: use checkout once and after that update again" ;\
+	   echo "for cvsup: checkout kde-common from cvsup and" ;\
+	   echo "   link kde-common/admin to ./admin" ;\
+	   exit 1 ;\
+	fi
+	$(MAKE) -f admin/Makefile.common cvs
+
+.SILENT:
+

Property changes on: Makefile.cvs
___________________________________________________________________
Name: svn:keywords
   + Author Date Id Revision


Property changes on: .
___________________________________________________________________
Name: svn:externals
   + admin https://svn.kde.org/home/kde/branches/KDE/3.4/kde-common/admin


