--- kdenetwork-3.0.4/kmailcvt/kmailcvt.desktop.orig	Tue Nov 19 00:23:48 2002
+++ kdenetwork-3.0.4/kmailcvt/kmailcvt.desktop	Tue Nov 19 00:25:48 2002
@@ -1,6 +1,6 @@
 [KDE Desktop Entry]
 Encoding=UTF-8
-Name=KMail/KAB Import Tool
+Name=KMail/KAB import tool
 Name[af]=Kpos/Kab In voer Program
 Name[ar]=أداة استيراد KMail/دفتر العناوين
 Name[bg]=Импорт на Инструменти в KMail/KAB
@@ -23,7 +23,7 @@
 Name[lv]=KMail/KAB Importa Rīks
 Name[nb]=KMail-/KAB-importverktøy
 Name[nn]=KMail-/KAB-importverktøy
-Name[pl]=Narzędzie do importu książek adresowych KAB
+Name[pl]=Import do KMail/KAB
 Name[pt]=Ferramenta de Importação do KMail/KAB
 Name[pt_BR]=Importação - Kmail/KAB
 Name[ro]=Utilitar importare KMail/KAB
@@ -42,6 +42,8 @@
 Name[zh_CN]=KMail/KAB 导入工具
 Name[zh_TW]=KMail/KAB 匯入工具
 Name[zu]=Ithuluzi Lohwebo lwangaphakathi lwe KMail/KAB
+Comment=KMail/KAB adressbook import tool
+Comment[pl]=Narzędzie do importu książek adresowych KAB
 Type=Application
 Exec=kmailcvt -caption "%c" %i %m  
 Icon=kmailcvt.png
--- kdenetwork-3.0.4/kppp/logview/kppplogview.desktop.orig	Tue Nov 19 00:29:35 2002
+++ kdenetwork-3.0.4/kppp/logview/kppplogview.desktop	Tue Nov 19 00:30:19 2002
@@ -25,7 +25,7 @@
 Name[lv]=KPPP - Žurnāla Skatītājs
 Name[nb]=KPPP - loggvisning
 Name[nn]=KPPP - loggvising
-Name[pl]=KPPP - Przeglądarka logów
+Name[pl]=KPPP - logi
 Name[pt]=KPPP - Visualizador dos Registos
 Name[pt_BR]=KPPP - Visualisador de Logs
 Name[ro]=Jurnal conectare Internet
@@ -44,6 +44,7 @@
 Name[zh_CN]=KPPP - 记录查看
 Name[zh_TW]=KPPP - 紀錄檔檢視器
 Name[zu]=KPPP - Umboniso Wongeniso
+Comment[pl]=KPPP - Przeglądarka logów
 MimeType=
 Exec=kppplogview
 Icon=kppp
