--- kdenetwork-3.0.3/lanbrowsing/kio_lan/kio_lan.cpp--	2002-09-06 07:32:57.000000000 +0200
+++ kdenetwork-3.0.3/lanbrowsing/kio_lan/kio_lan.cpp	2002-09-06 07:35:23.000000000 +0200
@@ -132,7 +132,14 @@ int LANProtocol::readDataFromServer()
 int LANProtocol::lanReadDataFromServer()
 {
    kdDebug(7101)<<"LANProtocol::lanReadDataFromServer() host: "<<m_currentHost<<" port: "<<m_port<<endl;
-   connectToHost(m_currentHost.latin1(), m_port);
+
+   if (!connectToHost(m_currentHost.latin1(), m_port, false))
+   {
+      error(ERR_SLAVE_DEFINED, i18n("<qt>The lisa daemon does not appear to be running.<p>"
+                                    "In order to use the LAN Browser the lisa daemon must be "
+                                    "installed and activated by the system administrator."));
+      return 0;
+   }
    kdDebug(7101)<<"LANProtocol::lanReadDataFromServer() connected"<<endl;
 
    int receivedBytes(0);
