https://bugs.pld-linux.org/?do=details&id=5683

diff -ur kdenetwork-3.5.1/kopete/protocols/groupwise/gwaccount.cpp kdenetwork-3.5.1-qt-plugin/kopete/protocols/groupwise/gwaccount.cpp
--- kdenetwork-3.5.1/kopete/protocols/groupwise/gwaccount.cpp	2005-11-19 12:47:05.000000000 +0200
+++ kdenetwork-3.5.1-qt-plugin/kopete/protocols/groupwise/gwaccount.cpp	2006-02-15 11:14:12.000000000 +0200
@@ -242,7 +242,9 @@
 	if (!sslPossible)
 	{
 		KMessageBox::queuedMessageBox(Kopete::UI::Global::mainWidget (), KMessageBox::Error,
-							i18n ("SSL support could not be initialized for account %1. This is most likely because the QCA TLS plugin is not installed on your system.").
+							i18n ("SSL support could not be initialized for account %1. This is most likely because the QCA TLS plugin is not installed on your system. "
+									"You need to install qt-plugin-qca-tls RPM package on PLD Linux."
+								).
 							arg(myself()->contactId()),
 							i18n ("GroupWise SSL Error"));
 		return;
diff -ur kdenetwork-3.5.1/kopete/protocols/jabber/jabberaccount.cpp kdenetwork-3.5.1-qt-plugin/kopete/protocols/jabber/jabberaccount.cpp
--- kdenetwork-3.5.1/kopete/protocols/jabber/jabberaccount.cpp	2006-01-19 18:42:57.000000000 +0200
+++ kdenetwork-3.5.1-qt-plugin/kopete/protocols/jabber/jabberaccount.cpp	2006-02-15 11:13:49.000000000 +0200
@@ -366,7 +366,9 @@
 		case JabberClient::NoTLS:
 			// no SSL support, at the connecting stage this means the problem is client-side
 			KMessageBox::queuedMessageBox(Kopete::UI::Global::mainWidget (), KMessageBox::Error,
-								i18n ("SSL support could not be initialized for account %1. This is most likely because the QCA TLS plugin is not installed on your system.").
+								i18n ("SSL support could not be initialized for account %1. This is most likely because the QCA TLS plugin is not installed on your system. "
+									"You need to install qt-plugin-qca-tls RPM package on PLD Linux."
+									).
 								arg(myself()->contactId()),
 								i18n ("Jabber SSL Error"));
 			break;
diff -ur kdenetwork-3.5.1/kopete/protocols/jabber/ui/jabberregisteraccount.cpp kdenetwork-3.5.1-qt-plugin/kopete/protocols/jabber/ui/jabberregisteraccount.cpp
--- kdenetwork-3.5.1/kopete/protocols/jabber/ui/jabberregisteraccount.cpp	2006-01-19 18:42:57.000000000 +0200
+++ kdenetwork-3.5.1-qt-plugin/kopete/protocols/jabber/ui/jabberregisteraccount.cpp	2006-02-15 11:13:55.000000000 +0200
@@ -264,7 +264,9 @@
 		case JabberClient::NoTLS:
 			// no SSL support, at the connecting stage this means the problem is client-side
 			KMessageBox::queuedMessageBox(Kopete::UI::Global::mainWidget (), KMessageBox::Error,
-								i18n ("SSL support could not be initialized for account %1. This is most likely because the QCA TLS plugin is not installed on your system.").
+								i18n ("SSL support could not be initialized for account %1. This is most likely because the QCA TLS plugin is not installed on your system. "
+									"You need to install qt-plugin-qca-tls RPM package on PLD Linux."
+									).
 								arg ( mMainWidget->leJID->text () ),
 								i18n ("Jabber SSL Error"));
 			break;
