--- kopete/protocols/oscar/liboscar/client.cpp	2006-08-12 02:51:26.000000000 +0200
+++ kopete/protocols/oscar/liboscar/client.cpp-new	2006-11-03 04:31:10.020580904 +0100
@@ -340,6 +340,7 @@
 		//we've finished logging in. start the services setup
 		kdDebug(OSCAR_RAW_DEBUG) << k_funcinfo << "stage two done. setting up services" << endl;
 		initializeStaticTasks();
+		d->ownStatusTask->go();
 		ServiceSetupTask* ssTask = new ServiceSetupTask( d->connections.defaultConnection()->rootTask() );
 		connect( ssTask, SIGNAL( finished() ), this, SLOT( serviceSetupFinished() ) );
 		ssTask->go( true ); //fire and forget
