diff -urN kdenetwork-3.0.3.orig/kpf/src/DirectoryLister.cpp kdenetwork-3.0.3/kpf/src/DirectoryLister.cpp
--- kdenetwork-3.0.3.orig/kpf/src/DirectoryLister.cpp	Mon Jun 24 14:06:11 2002
+++ kdenetwork-3.0.3/kpf/src/DirectoryLister.cpp	Sat Oct 12 14:47:01 2002
@@ -255,13 +255,6 @@
 
       html_stream << "<td class=\"" << td_class << "\">";
 
-      html_stream << "<img src=\"/?icon=";
-      if (fi->fileName() != "..")
-          html_stream << KMimeType::iconForURL(KURL(root + path + fi->fileName()));
-      else
-          html_stream << "folder_open" << endl;
-      html_stream << "\" alt=\"\">&nbsp;";
-
       QString item_class = QString((fi->isDir()) ? "direntry" : "fileentry");
       html_stream << "<a href=\"" + QStyleSheet::escape(path + fi->fileName()) + "\" class=\"" << item_class << "\">";
 
diff -urN kdenetwork-3.0.3.orig/kpf/src/Resource.cpp kdenetwork-3.0.3/kpf/src/Resource.cpp
--- kdenetwork-3.0.3.orig/kpf/src/Resource.cpp	Mon Jun 24 14:06:11 2002
+++ kdenetwork-3.0.3/kpf/src/Resource.cpp	Sat Oct 12 14:46:26 2002
@@ -27,7 +27,6 @@
 #include <qfileinfo.h>
 #include <qregexp.h>
 #include <kglobal.h>
-#include <kiconloader.h>
 
 #include <kmimemagic.h>
 
@@ -94,14 +93,7 @@
       d->root += '/';
 
 
-    // Was an icon requested ?
-    QRegExp iconExp("^?icon=(.*)$");
-
-    if (iconExp.search(d->path) != -1) {
-      d->root = "";
-      d->path = KGlobal::iconLoader()->iconPath(iconExp.cap(1), KIcon::Small);
-    }
-    else if (d->path.right(1) == "/")
+    if (d->path.right(1) == "/")
     {
       // A directory was requested 
       kpfDebug << "Directory requested" << endl;
